# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# 
# Translators:
# András Bögöly <ebogoly@gmail.com>, 2011-2013.
#   <ptr@ulx.hu>, 2012.
msgid ""
msgstr ""
"Project-Id-Version: Yum\n"
"Report-Msgid-Bugs-To: http://bugzilla.redhat.com/\n"
"POT-Creation-Date: 2013-01-30 09:08-0500\n"
"PO-Revision-Date: 2013-02-27 23:33+0000\n"
"Last-Translator: András Bögöly <ebogoly@gmail.com>\n"
"Language-Team: Hungarian (http://www.transifex.com/projects/p/yum/language/hu/)\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Language: hu\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"

#: ../callback.py:45 ../output.py:1502 ../yum/rpmtrans.py:73
msgid "Updating"
msgstr "Frissítés"

#: ../callback.py:46 ../yum/rpmtrans.py:74
msgid "Erasing"
msgstr "Törlés"

#: ../callback.py:47 ../callback.py:48 ../callback.py:50 ../output.py:1501
#: ../output.py:2922 ../yum/rpmtrans.py:75 ../yum/rpmtrans.py:76
#: ../yum/rpmtrans.py:78
msgid "Installing"
msgstr "Telepítés"

#: ../callback.py:49 ../callback.py:55 ../output.py:2379 ../yum/rpmtrans.py:77
msgid "Obsoleted"
msgstr "Elévült"

#: ../callback.py:51 ../output.py:1670 ../output.py:2222 ../output.py:2386
msgid "Updated"
msgstr "Frissítve"

#: ../callback.py:52 ../output.py:2221
msgid "Erased"
msgstr "Törölve"

#: ../callback.py:53 ../callback.py:54 ../callback.py:56 ../output.py:1668
#: ../output.py:2221 ../output.py:2223 ../output.py:2894
msgid "Installed"
msgstr "Telepítve"

#: ../callback.py:142
msgid "No header - huh?"
msgstr "Hiányzik a fejléc - hm?"

#: ../callback.py:180
msgid "Repackage"
msgstr "Újracsomagolás"

#: ../callback.py:201
#, python-format
msgid "Error: invalid output state: %s for %s"
msgstr "Hiba: érvénytelen kimeneti állapot: %s %s"

#: ../callback.py:224
#, python-format
msgid "Erased: %s"
msgstr "Eltávolítva: %s"

#: ../callback.py:229 ../output.py:1503 ../output.py:2897
msgid "Removing"
msgstr "Eltávolítás"

#: ../callback.py:231 ../yum/rpmtrans.py:79
msgid "Cleanup"
msgstr "Tisztítás"

#: ../cli.py:122
#, python-format
msgid "Command \"%s\" already defined"
msgstr "A(z) \"%s\" nevű parancs már létezik"

#: ../cli.py:137
msgid "Setting up repositories"
msgstr "Tárolók beállítása"

#: ../cli.py:148
msgid "Reading repository metadata in from local files"
msgstr "Tárolók metaadatainak beolvasása helyi fájlokból"

#: ../cli.py:273 ../cli.py:277 ../utils.py:320
#, python-format
msgid "Config Error: %s"
msgstr "Beállítási hiba: %s"

#: ../cli.py:280 ../cli.py:2206 ../utils.py:323
#, python-format
msgid "Options Error: %s"
msgstr "Opció hiba: %s"

#: ../cli.py:327
#, python-format
msgid "  Installed: %s-%s at %s"
msgstr "  Telepítve : %s-%s at %s"

#: ../cli.py:329
#, python-format
msgid "  Built    : %s at %s"
msgstr "  Létrehozva: %s at %s"

#: ../cli.py:331
#, python-format
msgid "  Committed: %s at %s"
msgstr "  Hozzáadva: %s at %s"

#: ../cli.py:372
msgid "You need to give some command"
msgstr "Egy parancs megadása szükségeltetik"

#: ../cli.py:386
#, python-format
msgid "No such command: %s. Please use %s --help"
msgstr "Nincs ilyen parancs: %s. Kérjük használja a következőt: %s --help"

#: ../cli.py:444
msgid "Disk Requirements:\n"
msgstr "Szükséges hely:\n"

#: ../cli.py:446
#, python-format
msgid "  At least %dMB more space needed on the %s filesystem.\n"
msgid_plural "  At least %dMB more space needed on the %s filesystem.\n"
msgstr[0] ""
msgstr[1] "  Legalább %dMB-tal több helyre van szükség az alábbi fájlrendszeren: %s\n"

#. TODO: simplify the dependency errors?
#. Fixup the summary
#: ../cli.py:451
msgid ""
"Error Summary\n"
"-------------\n"
msgstr "Hiba összegzés\n-------------\n"

#: ../cli.py:472
msgid "Can't create lock file; exiting"
msgstr "Nem sikerült lezárni; kilépés..."

#: ../cli.py:479
msgid ""
"Another app is currently holding the yum lock; exiting as configured by "
"exit_on_lock"
msgstr "Egy másik alkalmazás használja a yum zárat, ezért kilépünk, ahogy be van állítva az exit_on_lock szerint"

#: ../cli.py:532
msgid "Trying to run the transaction but nothing to do. Exiting."
msgstr "A tranzakciót megpróbáltuk végrehajtani, de nincs semmi tennivaló. Kilépünk."

#: ../cli.py:577
msgid "future rpmdb ver mismatched saved transaction version,"
msgstr "jövőbeli rpmbd ver nem egyezik a mentett tranzakció verziójával,"

#: ../cli.py:579 ../yum/__init__.py:6523
msgid " ignoring, as requested."
msgstr " kihagyva, ahogy kértük."

#: ../cli.py:582 ../yum/__init__.py:6526 ../yum/__init__.py:6673
msgid " aborting."
msgstr "megszakítás."

#: ../cli.py:588
msgid "Exiting on user Command"
msgstr "Felhasználói parancs miatt kilépés."

#: ../cli.py:592
msgid "Downloading Packages:"
msgstr "Csomagok letöltése:"

#: ../cli.py:597
msgid "Error Downloading Packages:\n"
msgstr "Hiba ezen csomagok letöltésekor:\n"

#: ../cli.py:616 ../yum/__init__.py:6215
msgid "Running Transaction Check"
msgstr "Tranzakció-ellenőrzés futtatása"

#: ../cli.py:625 ../yum/__init__.py:6224
msgid "ERROR You need to update rpm to handle:"
msgstr "HIBA frissíteni kell az rpmet, hogy kezeljük:"

#: ../cli.py:627 ../yum/__init__.py:6227
msgid "ERROR with transaction check vs depsolve:"
msgstr "HIBA a tranzakció ellenőrzés- és függőségfeloldásnál:"

#: ../cli.py:633
msgid "RPM needs to be updated"
msgstr "RPM frissítésére van szükség"

#: ../cli.py:634
#, python-format
msgid "Please report this error in %s"
msgstr "Kérem jelentse ezt a hibát a következőben: %s"

#: ../cli.py:640
msgid "Running Transaction Test"
msgstr "Tranzakció teszt futtatása"

#: ../cli.py:652
msgid "Transaction Check Error:\n"
msgstr "Tranzakció hibaellenőrzése:\n"

#: ../cli.py:659
msgid "Transaction Test Succeeded"
msgstr "A Tranzakció teszt sikeres"

#: ../cli.py:691
msgid "Running Transaction"
msgstr "Tranzakció futtatása"

#: ../cli.py:724
msgid ""
"Refusing to automatically import keys when running unattended.\n"
"Use \"-y\" to override."
msgstr "Felügyelet nélküli futás közben kulcsok nem importálhatóak.\nHasználjuk az \"-y\" kapcsolót a felülbíráláshoz."

#: ../cli.py:743 ../cli.py:786
msgid "  * Maybe you meant: "
msgstr "* Lehet, hogy erre gondolt:"

#: ../cli.py:769 ../cli.py:777
#, python-format
msgid "Package(s) %s%s%s available, but not installed."
msgstr "A(z) %s%s%s elérhető, de nincs telepítve."

#: ../cli.py:783 ../cli.py:891 ../cli.py:1158
#, python-format
msgid "No package %s%s%s available."
msgstr "Nincs ilyen csomag: %s%s%s."

#: ../cli.py:871 ../cli.py:881 ../cli.py:1101 ../cli.py:1111
#, python-format
msgid "Bad %s argument %s."
msgstr "Hibás %s argumentum %s."

#: ../cli.py:900 ../yumcommands.py:3331
#, python-format
msgid "%d package to install"
msgid_plural "%d packages to install"
msgstr[0] "%d csomag telepítésre kerül"
msgstr[1] "%d csomag telepítésre kerül"

#: ../cli.py:903 ../cli.py:904 ../cli.py:1169 ../cli.py:1170 ../cli.py:1224
#: ../cli.py:1225 ../cli.py:1260 ../yumcommands.py:323 ../yumcommands.py:3419
msgid "Nothing to do"
msgstr "Nincs tennivaló"

#: ../cli.py:953
#, python-format
msgid "%d package marked for Update"
msgid_plural "%d packages marked for Update"
msgstr[0] "%d csomag frissítésre kerül"
msgstr[1] "%d csomag frissítésre kerül"

#: ../cli.py:955
msgid "No Packages marked for Update"
msgstr "Nincs csomag kijelölve frissítésre"

#: ../cli.py:1067
#, python-format
msgid "%d package marked for Distribution Synchronization"
msgid_plural "%d packages marked for Distribution Synchronization"
msgstr[0] "%d csomag kijelölve Disztribúció Szinkronizációhoz"
msgstr[1] "%d csomag kijelölve Disztribúció Szinkronizációhoz"

#: ../cli.py:1069
msgid "No Packages marked for Distribution Synchronization"
msgstr "Nincs csomag kijelölve Disztribúció Szinkronizáláshoz"

#: ../cli.py:1124
#, python-format
msgid "%d package marked for removal"
msgid_plural "%d packages marked for removal"
msgstr[0] "%d csomag eltávolításra kerül"
msgstr[1] "%d csomag eltávolításra kerül"

#: ../cli.py:1126
msgid "No Packages marked for removal"
msgstr "Nincs csomag kijelölve eltávolításra"

#: ../cli.py:1166
#, python-format
msgid "%d package to downgrade"
msgid_plural "%d packages to downgrade"
msgstr[0] "%d csomag visszaállításra"
msgstr[1] "%d csomag visszaállításra"

#: ../cli.py:1207
#, python-format
msgid " (from %s)"
msgstr "(ebből: %s)"

#: ../cli.py:1208
#, python-format
msgid "Installed package %s%s%s%s not available."
msgstr "A következő telepített csomag nem elérhető: %s%s%s%s"

#: ../cli.py:1221
#, python-format
msgid "%d package to reinstall"
msgid_plural "%d packages to reinstall"
msgstr[0] "%d csomag újra lesz telepítve"
msgstr[1] "%d csomag újra lesz telepítve"

#: ../cli.py:1246
msgid "No Packages Provided"
msgstr "Nem találhatóak csomagok"

#: ../cli.py:1259
msgid "Package(s) to install"
msgstr "Telepítendő csomagok"

#: ../cli.py:1367
#, python-format
msgid "N/S Matched: %s"
msgstr "N/S Megegyezett: %s"

#: ../cli.py:1384
#, python-format
msgid "  Name and summary matches %sonly%s, use \"search all\" for everything."
msgstr " Csak a név és a leírás %segyezik%s, használja a \"search all\" parancsot továbbiak kereséséhez."

#: ../cli.py:1386
#, python-format
msgid ""
"  Full name and summary matches %sonly%s, use \"search all\" for everything."
msgstr " Csak a teljes név és a leírás %segyezik%s, használja a \"search all\" parancsot továbbiak kereséséhez."

#: ../cli.py:1404
#, python-format
msgid "Matched: %s"
msgstr "Találat: %s"

#: ../cli.py:1411
#, python-format
msgid "  Name and summary matches %smostly%s, use \"search all\" for everything."
msgstr " Csak a név és a leírás %segyezik%s többnyire, használja a \"search all\" parancsot továbbiak kereséséhez."

#: ../cli.py:1415
#, python-format
msgid "Warning: No matches found for: %s"
msgstr "Figyelem: Nincs találat a következőnél: %s"

#: ../cli.py:1418
msgid "No Matches found"
msgstr "Nincsenek találatok"

#: ../cli.py:1536
#, python-format
msgid ""
"Error: No Packages found for:\n"
"  %s"
msgstr "Hiba: Nem található csomag a következőre:\n  %s"

#: ../cli.py:1572
msgid "Cleaning repos: "
msgstr "Tárolók tisztítása"

#: ../cli.py:1577
msgid "Cleaning up Everything"
msgstr "Minden tisztítása"

#: ../cli.py:1593
msgid "Cleaning up Headers"
msgstr "Fejlécek tisztítása"

#: ../cli.py:1596
msgid "Cleaning up Packages"
msgstr "Csomagok tisztítása"

#: ../cli.py:1599
msgid "Cleaning up xml metadata"
msgstr "xml metaadat tisztítása"

#: ../cli.py:1602
msgid "Cleaning up database cache"
msgstr "Adatbázis-gyorsítótár tisztítása"

#: ../cli.py:1605
msgid "Cleaning up expire-cache metadata"
msgstr "Elévülési gyorsítótár tisztítása"

#: ../cli.py:1608
msgid "Cleaning up cached rpmdb data"
msgstr "Gyorsítótárazott rpmdb adatok tisztítása"

#: ../cli.py:1611
msgid "Cleaning up plugins"
msgstr "Bővítmények tisztítása"

#: ../cli.py:1727
msgid "Installed Environment Groups:"
msgstr "Telepített Környezeti Csoportok:"

#: ../cli.py:1728
msgid "Available Environment Groups:"
msgstr "Elérhető Környezeti Csoportok:"

#: ../cli.py:1735
msgid "Installed Groups:"
msgstr "Telepített csoportok:"

#: ../cli.py:1742
msgid "Installed Language Groups:"
msgstr "Telepített nyelvi csoportok:"

#: ../cli.py:1749
msgid "Available Groups:"
msgstr "Elérhető csoportok:"

#: ../cli.py:1756
msgid "Available Language Groups:"
msgstr "Elérhető nyelvi csoportok:"

#: ../cli.py:1759
#, python-format
msgid "Warning: No Environments/Groups match: %s"
msgstr "Figyelem: Nincs Környezeti/Csoporti egyezés: %s"

#: ../cli.py:1763
msgid "Done"
msgstr "Kész"

#: ../cli.py:1818
#, python-format
msgid "Warning: Group/Environment %s does not exist."
msgstr "Figyelem: A következő Környezet/Csoport nem létezik: %s"

#: ../cli.py:1859
#, python-format
msgid "Warning: Environment %s does not exist."
msgstr "Figyelem: A következő Környezet nem létezik: %s"

#: ../cli.py:1873 ../cli.py:1879 ../yum/__init__.py:4254
#, python-format
msgid "Warning: Group %s does not exist."
msgstr "Figyelem: Az alábbi csoport nem létezik: %s"

#: ../cli.py:1883
msgid "No packages in any requested group available to install or update"
msgstr "Nincs telepítendő vagy frissítendő csomag a kiválasztott csoportokból"

#: ../cli.py:1885
#, python-format
msgid "%d package to Install"
msgid_plural "%d packages to Install"
msgstr[0] "%d csomag telepítésre kerül"
msgstr[1] "%d csomag telepítésre kerül"

#: ../cli.py:1919 ../yum/__init__.py:3536 ../yum/__init__.py:3766
#: ../yum/__init__.py:3824
#, python-format
msgid "No Environment named %s exists"
msgstr "A(z) %s elnevezésű környezet nem létezik"

#: ../cli.py:1935 ../yum/__init__.py:4282
#, python-format
msgid "No group named %s exists"
msgstr "Nincs ilyen csoport: %s"

#: ../cli.py:1945
msgid "No packages to remove from groups"
msgstr "Nincs szükség csomag eltávolítására csoportokból"

#: ../cli.py:1947 ../yumcommands.py:3351
#, python-format
msgid "%d package to remove"
msgid_plural "%d packages to remove"
msgstr[0] "%d csomag eltávolításra kerül"
msgstr[1] "%d csomag eltávolításra kerül"

#: ../cli.py:1988
#, python-format
msgid "Package %s is already installed, skipping"
msgstr "A(z) %s nevű csomag már telepítve van, ezért továbblépünk."

#: ../cli.py:1999
#, python-format
msgid "Discarding non-comparable pkg %s.%s"
msgstr "Megválunk a következőtől: pkg %s.%s"

#. we've not got any installed that match n or n+a
#: ../cli.py:2025
#, python-format
msgid "No other %s installed, adding to list for potential install"
msgstr "Nincs más %s telepítve, ezért hozzáadjuk a listához."

#: ../cli.py:2045
msgid "Plugin Options"
msgstr "Bővítmény beállítások"

#: ../cli.py:2057
#, python-format
msgid "Command line error: %s"
msgstr "Parancssori hiba a következőben: %s"

#: ../cli.py:2079
#, python-format
msgid ""
"\n"
"\n"
"%s: %s option requires an argument"
msgstr "\n\n%s: %s opciónak szüksége van egy argumentumra"

#: ../cli.py:2147
msgid "--color takes one of: auto, always, never"
msgstr "--Szín egyikét veszi fel: auto, mindig, soha"

#. We have a relative installroot ... haha
#: ../cli.py:2218
#, python-format
msgid "--installroot must be an absolute path: %s"
msgstr "A(z) --instalroot -nak abszolút elérési útnak kell lennie: %s"

#: ../cli.py:2272
msgid "show this help message and exit"
msgstr "ezen segítség mutatása"

#: ../cli.py:2276
msgid "be tolerant of errors"
msgstr "a hibák figyelmen kívül hagyása"

#: ../cli.py:2279
msgid "run entirely from system cache, don't update cache"
msgstr "futtatás a rendszer gyorsítótárából anélkül, hogy a csomagok vagy tárolóinformációk frissítve lennének"

#: ../cli.py:2282
msgid "config file location"
msgstr "konfigurációs fájl elérésí útja"

#: ../cli.py:2285
msgid "maximum command wait time"
msgstr "leghosszabb várakozási idő meghatározása"

#: ../cli.py:2287
msgid "debugging output level"
msgstr "a hibakeresés szintje"

#: ../cli.py:2291
msgid "show duplicates, in repos, in list/search commands"
msgstr "ismétlődések mutatása a tárolókban és a keresési parancsokban"

#: ../cli.py:2296
msgid "error output level"
msgstr "hiba kimeneti szintje"

#: ../cli.py:2299
msgid "debugging output level for rpm"
msgstr "rpm hibakereső szint"

#: ../cli.py:2302
msgid "quiet operation"
msgstr "csendes működés"

#: ../cli.py:2304
msgid "verbose operation"
msgstr "visszajelzés működés közben"

#: ../cli.py:2306
msgid "answer yes for all questions"
msgstr "igennel válaszol minden felmerülő kérdésre"

#: ../cli.py:2308
msgid "answer no for all questions"
msgstr "összes kérdésre a válasz nem"

#: ../cli.py:2312
msgid "show Yum version and exit"
msgstr "Yum verziójának mutatása és kilépés"

#: ../cli.py:2313
msgid "set install root"
msgstr "telepítési root beállítása"

#: ../cli.py:2317
msgid "enable one or more repositories (wildcards allowed)"
msgstr "egy vagy több tároló engedélyezése (helyettesítő karakterek megengedettek)"

#: ../cli.py:2321
msgid "disable one or more repositories (wildcards allowed)"
msgstr "egy vagy több tároló tiltása (helyettesítő karakterek megengedettek)"

#: ../cli.py:2324
msgid "exclude package(s) by name or glob"
msgstr "csomagok kizárása név vagy glob szerint"

#: ../cli.py:2326
msgid "disable exclude from main, for a repo or for everything"
msgstr "kizárás letiltása alapkomponensre, tárolókra ill. bármire"

#: ../cli.py:2329
msgid "enable obsoletes processing during updates"
msgstr "elavult komponensek feldolgozásának engedélyezése"

#: ../cli.py:2331
msgid "disable Yum plugins"
msgstr "Yum bővítmények tiltása"

#: ../cli.py:2333
msgid "disable gpg signature checking"
msgstr "gpg-aláírás ellenőrzés kihagyása"

#: ../cli.py:2335
msgid "disable plugins by name"
msgstr "bővítmények tiltása név szerint"

#: ../cli.py:2338
msgid "enable plugins by name"
msgstr "bővítmények engedélyezése név szerint"

#: ../cli.py:2341
msgid "skip packages with depsolving problems"
msgstr "azon csomagok átugrása, melyeknek függőségi problémáik vannak"

#: ../cli.py:2343
msgid "control whether color is used"
msgstr "szín használatának befolyásolása"

#: ../cli.py:2345
msgid "set value of $releasever in yum config and repo files"
msgstr "az alábbi érték beállítása a yum tároló és konfigurációs fájljaiban: $releasever"

#: ../cli.py:2347
msgid "don't update, just download"
msgstr "ne frissítsen, csak töltse le"

#: ../cli.py:2349
msgid "specifies an alternate directory to store packages"
msgstr "egy alternatív könyvtár meghatározása a csomagok tárolásához"

#: ../cli.py:2351
msgid "set arbitrary config and repo options"
msgstr "tetszőleges konfiguráció- és tárolóbeállítások"

#: ../output.py:458
msgid "Jan"
msgstr "jan."

#: ../output.py:458
msgid "Feb"
msgstr "feb."

#: ../output.py:458
msgid "Mar"
msgstr "márc."

#: ../output.py:458
msgid "Apr"
msgstr "ápr."

#: ../output.py:458
msgid "May"
msgstr "máj."

#: ../output.py:458
msgid "Jun"
msgstr "jún."

#: ../output.py:459
msgid "Jul"
msgstr "júl."

#: ../output.py:459
msgid "Aug"
msgstr "aug."

#: ../output.py:459
msgid "Sep"
msgstr "szept."

#: ../output.py:459
msgid "Oct"
msgstr "okt."

#: ../output.py:459
msgid "Nov"
msgstr "nov."

#: ../output.py:459
msgid "Dec"
msgstr "dec."

#: ../output.py:473
msgid "Trying other mirror."
msgstr "Próbálkozás másik tükörszerverről."

#: ../output.py:816
#, python-format
msgid "Name        : %s%s%s"
msgstr "Név         : %s%s%s"

#: ../output.py:817
#, python-format
msgid "Arch        : %s"
msgstr "Arch        : %s"

#: ../output.py:819
#, python-format
msgid "Epoch       : %s"
msgstr "Epoch       : %s"

#: ../output.py:820
#, python-format
msgid "Version     : %s"
msgstr "Verzió      : %s"

#: ../output.py:821
#, python-format
msgid "Release     : %s"
msgstr "Kiadás      : %s"

#: ../output.py:822
#, python-format
msgid "Size        : %s"
msgstr "Méret       : %s"

#: ../output.py:823 ../output.py:1329
#, python-format
msgid "Repo        : %s"
msgstr "Tároló      : %s"

#: ../output.py:825
#, python-format
msgid "From repo   : %s"
msgstr "Tárolóból   : %s"

#: ../output.py:827
#, python-format
msgid "Committer   : %s"
msgstr "Beküldő     : %s"

#: ../output.py:828
#, python-format
msgid "Committime  : %s"
msgstr "Beküldési idő: %s"

#: ../output.py:829
#, python-format
msgid "Buildtime   : %s"
msgstr "Kiállítási idő : %s"

#: ../output.py:831
#, python-format
msgid "Install time: %s"
msgstr "Telepítési idő: %s"

#: ../output.py:839
#, python-format
msgid "Installed by: %s"
msgstr "Telepítette: %s"

#: ../output.py:846
#, python-format
msgid "Changed by  : %s"
msgstr "Módosította  : %s"

#: ../output.py:847
msgid "Summary     : "
msgstr "Összegzés   : "

#: ../output.py:849 ../output.py:1345
#, python-format
msgid "URL         : %s"
msgstr "URL         : %s"

#: ../output.py:850
msgid "License     : "
msgstr "Licenc      : "

#: ../output.py:851 ../output.py:1342
msgid "Description : "
msgstr "Leírás      : "

#: ../output.py:969
msgid "y"
msgstr "y"

#: ../output.py:969
msgid "yes"
msgstr "igen"

#: ../output.py:970
msgid "n"
msgstr "n"

#: ../output.py:970
msgid "no"
msgstr "nem"

#: ../output.py:974
msgid "Is this ok [y/N]: "
msgstr "Ez így jó? [y/N]:"

#: ../output.py:1097
#, python-format
msgid ""
"\n"
"Group: %s"
msgstr "\nCsoport: %s"

#: ../output.py:1101
#, python-format
msgid " Group-Id: %s"
msgstr " Csoport azonosító: %s"

#: ../output.py:1122 ../output.py:1169
#, python-format
msgid " Description: %s"
msgstr " Leírás: %s"

#: ../output.py:1124
#, python-format
msgid " Language: %s"
msgstr " Nyelv: %s"

#: ../output.py:1126
msgid " Mandatory Packages:"
msgstr " Szükséges csomagok:"

#: ../output.py:1127
msgid " Default Packages:"
msgstr " Alapértelmezett csomagok:"

#: ../output.py:1128
msgid " Optional Packages:"
msgstr " Opcionális csomagok:"

#: ../output.py:1129
msgid " Conditional Packages:"
msgstr " Feltételes csomagok:"

#: ../output.py:1147
msgid " Installed Packages:"
msgstr " Telepített csomagok:"

#: ../output.py:1157
#, python-format
msgid ""
"\n"
"Environment Group: %s"
msgstr "\nKörnyezeti csoport: %s"

#: ../output.py:1158
#, python-format
msgid " Environment-Id: %s"
msgstr "Környezet azonosító: %s"

#: ../output.py:1191
msgid " Mandatory Groups:"
msgstr "Kötelező Csoportok:"

#: ../output.py:1192
msgid " Optional Groups:"
msgstr "Opcionális Csoportok:"

#: ../output.py:1205
msgid " Installed Groups:"
msgstr "Telepített Csoportok:"

#: ../output.py:1217
#, python-format
msgid "package: %s"
msgstr "csomag: %s"

#: ../output.py:1219
msgid "  No dependencies for this package"
msgstr "  Ezen csomaghoz nincs függőség"

#: ../output.py:1224
#, python-format
msgid "  dependency: %s"
msgstr "  függőség: %s"

#: ../output.py:1226
msgid "   Unsatisfied dependency"
msgstr "   Teljesítetlen függőség"

#: ../output.py:1337
msgid "Matched from:"
msgstr "Találat a következőtől:"

#: ../output.py:1348
#, python-format
msgid "License     : %s"
msgstr "Licenc      : %s"

#: ../output.py:1351
#, python-format
msgid "Filename    : %s"
msgstr "Fájlnév     : %s"

#: ../output.py:1360
msgid "Provides    : "
msgstr "Ellátja : "

#: ../output.py:1363
msgid "Other       : "
msgstr "Egyéb       : "

#: ../output.py:1426
msgid "There was an error calculating total download size"
msgstr "Hiba történt a teljes letöltési méret számítása során"

#: ../output.py:1431
#, python-format
msgid "Total size: %s"
msgstr "Teljes méret: %s"

#: ../output.py:1433
#, python-format
msgid "Total download size: %s"
msgstr "Teljes letöltési méret: %s"

#: ../output.py:1436 ../output.py:1459 ../output.py:1463
#, python-format
msgid "Installed size: %s"
msgstr "Telepített méret: %s"

#: ../output.py:1454
msgid "There was an error calculating installed size"
msgstr "Hiba történt a telepített méret számítása során"

#: ../output.py:1504
msgid "Reinstalling"
msgstr "Újratelepítés"

#: ../output.py:1505
msgid "Downgrading"
msgstr "Visszaállítás"

#: ../output.py:1506
msgid "Installing for dependencies"
msgstr "Telepítés a függőségeknek"

#: ../output.py:1507
msgid "Updating for dependencies"
msgstr "Frissítés a függőségeknek"

#: ../output.py:1508
msgid "Removing for dependencies"
msgstr "Eltávolítás a függőségeknek"

#: ../output.py:1515 ../output.py:1576 ../output.py:1672
msgid "Skipped (dependency problems)"
msgstr "Kihagyva (függőségi problémák)"

#: ../output.py:1517 ../output.py:1577 ../output.py:2223
msgid "Not installed"
msgstr "Nem telepítve"

#: ../output.py:1518 ../output.py:1578
msgid "Not available"
msgstr "Nem elérhető"

#: ../output.py:1540 ../output.py:1588 ../output.py:1604 ../output.py:2581
msgid "Package"
msgid_plural "Packages"
msgstr[0] ""
msgstr[1] "Csomag"

#: ../output.py:1540
msgid "Arch"
msgstr "Arch"

#: ../output.py:1541
msgid "Version"
msgstr "Verzió"

#: ../output.py:1541
msgid "Repository"
msgstr "Tároló"

#: ../output.py:1542
msgid "Size"
msgstr "Méret"

#: ../output.py:1554
#, python-format
msgid "     replacing  %s%s%s.%s %s\n"
msgstr "     lecserélés  %s%s%s.%s %s\n"

#: ../output.py:1563
#, python-format
msgid ""
"\n"
"Transaction Summary\n"
"%s\n"
msgstr "\nTranzakció Összegzés\n%s\n"

#: ../output.py:1568 ../output.py:2376 ../output.py:2377
msgid "Install"
msgstr "Telepítés"

#: ../output.py:1570
msgid "Upgrade"
msgstr "Frissítés"

#: ../output.py:1572
msgid "Remove"
msgstr "Eltávolítás"

#: ../output.py:1574 ../output.py:2382
msgid "Reinstall"
msgstr "Újratelepítés"

#: ../output.py:1575 ../output.py:2383
msgid "Downgrade"
msgstr "Visszaállítás"

#: ../output.py:1606
msgid "Dependent package"
msgid_plural "Dependent packages"
msgstr[0] "Függőben lévő csomag"
msgstr[1] "Függőben lévő csomagok"

#: ../output.py:1666
msgid "Removed"
msgstr "Eltávolítva"

#: ../output.py:1667
msgid "Dependency Removed"
msgstr "Függőség Eltávolítva"

#: ../output.py:1669
msgid "Dependency Installed"
msgstr "Függőség Telepítve"

#: ../output.py:1671
msgid "Dependency Updated"
msgstr "Függőség Frissítve"

#: ../output.py:1673
msgid "Replaced"
msgstr "Lecserélve"

#: ../output.py:1674
msgid "Failed"
msgstr "Sikertelen"

#. Delta between C-c's so we treat as exit
#: ../output.py:1764
msgid "two"
msgstr "kettő"

#. For translators: This is output like:
#. Current download cancelled, interrupt (ctrl-c) again within two seconds
#. to exit.
#. Where "interrupt (ctrl-c) again" and "two" are highlighted.
#: ../output.py:1775
#, python-format
msgid ""
"\n"
" Current download cancelled, %sinterrupt (ctrl-c) again%s within %s%s%s seconds\n"
"to exit.\n"
msgstr "\nA jelenlegi letöltés megszakítva, %s megszakítás (ctrl-c) mégegyszer %s, %s%s%s másodperc alatt\na kilépéshez.\n"

#: ../output.py:1786
msgid "user interrupt"
msgstr "Felhasználó által megszakítva"

#: ../output.py:1812
msgid "Total"
msgstr "Összesen"

#: ../output.py:1834
msgid "I"
msgstr "I"

#: ../output.py:1835
msgid "O"
msgstr "O"

#: ../output.py:1836
msgid "E"
msgstr "E"

#: ../output.py:1837
msgid "R"
msgstr "R"

#: ../output.py:1838
msgid "D"
msgstr "D"

#: ../output.py:1839
msgid "U"
msgstr "U"

#: ../output.py:1853
msgid "<unset>"
msgstr "<üres>"

#: ../output.py:1854
msgid "System"
msgstr "Rendszer"

#: ../output.py:1923
#, python-format
msgid "Skipping merged transaction %d to %d, as it overlaps"
msgstr "Egyesített tranzakció kihagyása: %d, mivel fedi a %d tranzakciót."

#: ../output.py:1933 ../output.py:2125
msgid "No transactions"
msgstr "Nincsenek tranzakciók"

#: ../output.py:1958 ../output.py:2570 ../output.py:2660
msgid "Bad transaction IDs, or package(s), given"
msgstr "Rossz tranzakció azonosító vagy csomagnév lett megadva"

#: ../output.py:2007
msgid "Command line"
msgstr "Parancssor"

#: ../output.py:2009 ../output.py:2458
msgid "Login user"
msgstr "Felhasználó"

#. REALLY Needs to use columns!
#: ../output.py:2010 ../output.py:2579
msgid "ID"
msgstr "azonosító"

#: ../output.py:2012
msgid "Date and time"
msgstr "Dátum és idő"

#: ../output.py:2013 ../output.py:2460 ../output.py:2580
msgid "Action(s)"
msgstr "Művelet(ek)"

#: ../output.py:2014 ../output.py:2461
msgid "Altered"
msgstr "Változtatta"

#: ../output.py:2061
msgid "No transaction ID given"
msgstr "Nem lett megadva tranzakció azonosító"

#: ../output.py:2087 ../output.py:2526
msgid "Bad transaction ID given"
msgstr "Rossz tranzakció azonosító lett megadva"

#: ../output.py:2092
msgid "Not found given transaction ID"
msgstr "Nem található a megadott tranzakció azonosítója"

#: ../output.py:2100
msgid "Found more than one transaction ID!"
msgstr "Több, mint egy tranzakció azonosító található!"

#: ../output.py:2151 ../output.py:2534
msgid "No transaction ID, or package, given"
msgstr "Nem lett megadva tranzakció azonosító vagy csomag"

#: ../output.py:2222 ../output.py:2384
msgid "Downgraded"
msgstr "Visszaállítva"

#: ../output.py:2224
msgid "Older"
msgstr "Régebbi"

#: ../output.py:2224
msgid "Newer"
msgstr "Újabb"

#: ../output.py:2261 ../output.py:2263 ../output.py:2681
msgid "Transaction ID :"
msgstr "Tranzakció azonosító:"

#: ../output.py:2265 ../output.py:2683
msgid "Begin time     :"
msgstr "Kezdés ideje   :"

#: ../output.py:2268 ../output.py:2270
msgid "Begin rpmdb    :"
msgstr "rpmdb kezdete:"

#: ../output.py:2286
#, python-format
msgid "(%u seconds)"
msgstr "(%u másodperc)"

#: ../output.py:2288
#, python-format
msgid "(%u minutes)"
msgstr "(%u perc)"

#: ../output.py:2290
#, python-format
msgid "(%u hours)"
msgstr "(%u óra)"

#: ../output.py:2292
#, python-format
msgid "(%u days)"
msgstr "(%u nap)"

#: ../output.py:2293
msgid "End time       :"
msgstr "Befejezés ideje:"

#: ../output.py:2296 ../output.py:2298
msgid "End rpmdb      :"
msgstr "rpmdb befejezése:"

#: ../output.py:2301 ../output.py:2303
msgid "User           :"
msgstr "Felhasználó    :"

#: ../output.py:2307 ../output.py:2310 ../output.py:2312 ../output.py:2314
#: ../output.py:2316
msgid "Return-Code    :"
msgstr "Visszatérési érték:"

#: ../output.py:2307 ../output.py:2312
msgid "Aborted"
msgstr "Megszakítva"

#: ../output.py:2310
msgid "Failures:"
msgstr "Hibák:"

#: ../output.py:2314
msgid "Failure:"
msgstr "Hiba:"

#: ../output.py:2316
msgid "Success"
msgstr "Siker"

#: ../output.py:2321 ../output.py:2323 ../output.py:2712
msgid "Command Line   :"
msgstr "Parancssor     :"

#: ../output.py:2332
#, python-format
msgid "Additional non-default information stored: %d"
msgstr "További nem alapértelmezett információ lett eltárolva: %d"

#. This is _possible_, but not common
#: ../output.py:2337
msgid "Transaction performed with:"
msgstr "A tranzakció lezajlott a következővel:"

#: ../output.py:2341
msgid "Packages Altered:"
msgstr "Módosított csomagok:"

#: ../output.py:2345
msgid "Packages Skipped:"
msgstr "Kihagyott csomagok:"

#: ../output.py:2353
msgid "Rpmdb Problems:"
msgstr "Rpmdb hibák:"

#: ../output.py:2364
msgid "Scriptlet output:"
msgstr "Scriptlet kimenet:"

#: ../output.py:2370
msgid "Errors:"
msgstr "Hibák:"

#: ../output.py:2378
msgid "Dep-Install"
msgstr "Függőség-Telepítés"

#: ../output.py:2380
msgid "Obsoleting"
msgstr "Elavulttá tevés"

#: ../output.py:2381
msgid "Erase"
msgstr "Törlés"

#: ../output.py:2385
msgid "Update"
msgstr "Frissítés"

#: ../output.py:2459
msgid "Time"
msgstr "Időpont"

#: ../output.py:2485
msgid "Last day"
msgstr "1 napja"

#: ../output.py:2486
msgid "Last week"
msgstr "1 hete"

#: ../output.py:2487
msgid "Last 2 weeks"
msgstr "2 hete"

#. US default :p
#: ../output.py:2488
msgid "Last 3 months"
msgstr "3 hónapja"

#: ../output.py:2489
msgid "Last 6 months"
msgstr "6 hónapja"

#: ../output.py:2490
msgid "Last year"
msgstr "1 éve"

#: ../output.py:2491
msgid "Over a year ago"
msgstr "Több, mint egy éve"

#: ../output.py:2538
#, python-format
msgid "No Transaction %s found"
msgstr "Nem található a következő tranzakció: %s"

#: ../output.py:2544
msgid "Transaction ID:"
msgstr "Tranzakció azonosító:"

#: ../output.py:2545
msgid "Available additional history information:"
msgstr "Rendelkezésre álló további információ az előzményekről:"

#: ../output.py:2558
#, python-format
msgid "%s: No additional data found by this name"
msgstr "%s: Nem található további adat erre a névre"

#: ../output.py:2684
msgid "Package        :"
msgstr "Csomag        :"

#: ../output.py:2685
msgid "State          :"
msgstr "Állapot          :"

#: ../output.py:2688
msgid "Size           :"
msgstr "Méret           :"

#: ../output.py:2690
msgid "Build host     :"
msgstr "Build kiszolgáló:"

#: ../output.py:2693
msgid "Build time     :"
msgstr "Build időpont:"

#: ../output.py:2695
msgid "Packager       :"
msgstr "Csomagoló       :"

#: ../output.py:2697
msgid "Vendor         :"
msgstr "Szállító         :"

#: ../output.py:2699
msgid "License        :"
msgstr "Licensz        :"

#: ../output.py:2701
msgid "URL            :"
msgstr "URL            :"

#: ../output.py:2703
msgid "Source RPM     :"
msgstr "Forrás RPM     :"

#: ../output.py:2706
msgid "Commit Time    :"
msgstr "Beküldés időpont:"

#: ../output.py:2708
msgid "Committer      :"
msgstr "Beküldő   :"

#: ../output.py:2710
msgid "Reason         :"
msgstr "Ok         :"

#: ../output.py:2714
msgid "From repo      :"
msgstr "Tároló      :"

#: ../output.py:2718
msgid "Installed by   :"
msgstr "Telepítve általa   :"

#: ../output.py:2722
msgid "Changed by     :"
msgstr "Megváltoztatva általa     :"

#: ../output.py:2767
msgid "installed"
msgstr "telepítve"

#: ../output.py:2768
msgid "an update"
msgstr "egy frissítés"

#: ../output.py:2769
msgid "erased"
msgstr "törölve"

#: ../output.py:2770
msgid "reinstalled"
msgstr "újratelepítve"

#: ../output.py:2771
msgid "a downgrade"
msgstr "egy visszaállítás"

#: ../output.py:2772
msgid "obsoleting"
msgstr "visszaállítás"

#: ../output.py:2773
msgid "updated"
msgstr "frissítve"

#: ../output.py:2774
msgid "obsoleted"
msgstr "elavult"

#: ../output.py:2778
#, python-format
msgid "---> Package %s.%s %s:%s-%s will be %s"
msgstr "---> A(z) %s.%s %s:%s-%s csomaggal a következő történik: %s"

#: ../output.py:2789
msgid "--> Running transaction check"
msgstr "--> Tranzakció ellenőrzés futtatása"

#: ../output.py:2795
msgid "--> Restarting Dependency Resolution with new changes."
msgstr "--> Függőségvizsgálat újraindítása az új változásokkal."

#: ../output.py:2801
msgid "--> Finished Dependency Resolution"
msgstr "--> A függőségvizsgálat véget ért"

#: ../output.py:2814 ../output.py:2827
#, python-format
msgid "--> Processing Dependency: %s for package: %s"
msgstr "--> A(z) %s függőség feldolgozása a következő csomaghoz: %s"

#: ../output.py:2841
#, python-format
msgid "---> Keeping package: %s due to %s"
msgstr "---> A(z) %s Csomag megtartása a következő miatt: %s"

#: ../output.py:2850
#, python-format
msgid "--> Unresolved Dependency: %s"
msgstr "--> Nem talált függőség: %s"

#: ../output.py:2867
#, python-format
msgid "Package: %s"
msgstr "Csomag: %s"

#: ../output.py:2869
#, python-format
msgid ""
"\n"
"    Requires: %s"
msgstr "\n    Megkövetel: %s"

#: ../output.py:2878
#, python-format
msgid ""
"\n"
"    %s: %s (%s)"
msgstr "\n    %s: %s (%s)"

#: ../output.py:2883
#, python-format
msgid ""
"\n"
"        %s"
msgstr "\n        %s"

#: ../output.py:2885
msgid ""
"\n"
"        Not found"
msgstr "\n        Nem található"

#. These should be the only three things we care about:
#: ../output.py:2900
msgid "Updated By"
msgstr "Frissítette"

#: ../output.py:2901
msgid "Downgraded By"
msgstr "Visszaállította"

#: ../output.py:2902
msgid "Obsoleted By"
msgstr "Elavulttá tette"

#: ../output.py:2920
msgid "Available"
msgstr "Elérhető"

#: ../output.py:2955 ../output.py:2968
#, python-format
msgid "--> Processing Conflict: %s conflicts %s"
msgstr "--> Konfliktus feldolgozása: %s ellentétben áll: %s"

#: ../output.py:2974
msgid "--> Populating transaction set with selected packages. Please wait."
msgstr "--> A tranzakció feltöltése csomagokkal. Kis türelmet."

#: ../output.py:2983
#, python-format
msgid "---> Downloading header for %s to pack into transaction set."
msgstr "---> A(z) %s fejlécének betöltése, hogy a tranzakcióhoz hozzáadhassuk."

#. self.event(txmbr.name, count, len(base.tsInfo), count, )
#. (te_current*100L)/te_total
#: ../output.py:3248
msgid "Verifying"
msgstr "Ellenőrzés"

#: ../utils.py:123
msgid "Running"
msgstr "Fut"

#: ../utils.py:124
msgid "Sleeping"
msgstr "Alszik"

#: ../utils.py:125
msgid "Uninterruptible"
msgstr "Nem megszakítható"

#: ../utils.py:126
msgid "Zombie"
msgstr "Zombi"

#: ../utils.py:127
msgid "Traced/Stopped"
msgstr "Nyomozott/Megállított"

#: ../utils.py:128 ../yumcommands.py:2193
msgid "Unknown"
msgstr "Ismeretlen"

#: ../utils.py:153
msgid "  The other application is: PackageKit"
msgstr "  A másik alkalmazás: PackageKit"

#: ../utils.py:155
#, python-format
msgid "  The other application is: %s"
msgstr "  A másik alkalmazás: %s"

#: ../utils.py:158
#, python-format
msgid "    Memory : %5s RSS (%5sB VSZ)"
msgstr "    Memória : %5s RSS (%5sB VSZ)"

#: ../utils.py:163
#, python-format
msgid "    Started: %s - %s ago"
msgstr "    Elindítva: %s - %s"

#: ../utils.py:165
#, python-format
msgid "    State  : %s, pid: %d"
msgstr "    Állapot  : %s, pid: %d"

#: ../utils.py:194 ../yummain.py:43
msgid ""
"\n"
"\n"
"Exiting on user cancel"
msgstr "\n\nFelhasználó által megszakítva"

#: ../utils.py:206 ../yummain.py:49
msgid ""
"\n"
"\n"
"Exiting on Broken Pipe"
msgstr "\n\nKilépés megtört cső miatt (broken pipe)"

#: ../utils.py:208 ../yummain.py:51
#, python-format
msgid ""
"\n"
"\n"
"%s"
msgstr "\n\n%s"

#: ../utils.py:326
#, python-format
msgid "PluginExit Error: %s"
msgstr "Bővítmény-Összeomlási Hiba: %s"

#: ../utils.py:329
#, python-format
msgid "Yum Error: %s"
msgstr "Yum Hiba: %s"

#: ../utils.py:387 ../yummain.py:147 ../yummain.py:186
#, python-format
msgid "Error: %s"
msgstr "Hiba: %s"

#: ../utils.py:391 ../yummain.py:191
msgid " You could try using --skip-broken to work around the problem"
msgstr " Próbálja meg használni --skip kapcsolót a probléma elkerüléséhez"

#: ../utils.py:393 ../yummain.py:87
msgid " You could try running: rpm -Va --nofiles --nodigest"
msgstr " Próbálja meg futtatni: rpm -Va --nofiles --nodigest"

#: ../utils.py:400 ../yummain.py:157 ../yummain.py:199
#, python-format
msgid "Unknown Error(s): Exit Code: %d:"
msgstr "Ismeretlen Hiba: Kilépési Kód: %d:"

#: ../utils.py:406 ../yummain.py:205
msgid ""
"\n"
"Dependencies Resolved"
msgstr "\nFüggőségek Megtalálva"

#: ../utils.py:422 ../yummain.py:237
msgid "Complete!"
msgstr "Kész!"

#: ../yumcommands.py:42
msgid " Mini usage:\n"
msgstr " Használat:\n"

#: ../yumcommands.py:52
msgid "You need to be root to perform this command."
msgstr "Csak a root felhasználó futtathatja ezt a parancsot."

#: ../yumcommands.py:67
msgid ""
"\n"
"You have enabled checking of packages via GPG keys. This is a good thing. \n"
"However, you do not have any GPG public keys installed. You need to download\n"
"the keys for packages you wish to install and install them.\n"
"You can do that by running the command:\n"
"    rpm --import public.gpg.key\n"
"\n"
"\n"
"Alternatively you can specify the url to the key you would like to use\n"
"for a repository in the 'gpgkey' option in a repository section and yum \n"
"will install it for you.\n"
"\n"
"For more information contact your distribution or package provider.\n"
msgstr "\nÖn engedélyezte a csomagok GPG kulcs alapú ellenőrzését. Ez egy jó dolog.\nViszont nem található egy GPG kulcs sem, ezért le kéne tölteni\na csomagokhoz tartozó kulcsokat, és telepíteni őket.\nA következő parancs segítségével végezheti ezt el:\n    rpm --import public.gpg.key\n\n\nAkár megadhatja a használni kívánt url címét a kulcsnak a tárolóknál\na 'gpgkey' opció alatt, így a yum telepíteni fogja majd önnek.\n\nTovábbi információért lépjen kapcsolatba a disztribúciójával ill. a csomagszolgáltatójával.\n"

#: ../yumcommands.py:82
#, python-format
msgid "Problem repository: %s"
msgstr "Hibás tároló: %s"

#: ../yumcommands.py:96
#, python-format
msgid "Error: Need to pass a list of pkgs to %s"
msgstr "Hiba: Meg kell adni a csomagok listáját a következőhöz: %s"

#: ../yumcommands.py:114
#, python-format
msgid "Error: Need at least two packages to %s"
msgstr "Hiba: Legalább kettő csomag szükséges a következőhöz: %s"

#: ../yumcommands.py:129
#, python-format
msgid "Error: Need to pass a repoid. and command to %s"
msgstr "Hiba: Meg kell adni egy tároló azonosítót és további parancsokat a következőhöz: %s"

#: ../yumcommands.py:136 ../yumcommands.py:142
#, python-format
msgid "Error: Need to pass a single valid repoid. to %s"
msgstr "Hiba: Meg kell adni egy érvényes tárolóazonosítót a következőhöz: %s"

#: ../yumcommands.py:147
#, python-format
msgid "Error: Repo %s is not enabled"
msgstr "Hiba: A(z) %s tároló nincs engedélyezve"

#: ../yumcommands.py:164
msgid "Error: Need an item to match"
msgstr "Hiba: egy elemnek legalább egyeznie kell"

#: ../yumcommands.py:178
msgid "Error: Need a group or list of groups"
msgstr "Hiba: Szükség van legalább egy csoportra"

#: ../yumcommands.py:195
#, python-format
msgid "Error: clean requires an option: %s"
msgstr "Hiba: a clean parancsnak szüksége van egy opcióra: %s"

#: ../yumcommands.py:201
#, python-format
msgid "Error: invalid clean argument: %r"
msgstr "Hiba: helytelen clean argumentum: %r"

#: ../yumcommands.py:216
msgid "No argument to shell"
msgstr "Nincs argumentum a héj felé"

#: ../yumcommands.py:218
#, python-format
msgid "Filename passed to shell: %s"
msgstr "Fájlnév átadva a héjnak: %s"

#: ../yumcommands.py:222
#, python-format
msgid "File %s given as argument to shell does not exist."
msgstr "A(z) %s nevű fájl nem létező argumentumot adott át a héjnak."

#: ../yumcommands.py:228
msgid "Error: more than one file given as argument to shell."
msgstr "Hiba: több, mint egy fájl lett átadva argumentumként a héjnak."

#: ../yumcommands.py:247
msgid ""
"There are no enabled repos.\n"
" Run \"yum repolist all\" to see the repos you have.\n"
" You can enable repos with yum-config-manager --enable <repo>"
msgstr "Nincsenek engedélyezett tárolók.\nFuttassa a \"yum repolist all\" parancsot a tárolók listázásához.\nA következő paranccsal engedélyezhet tárolót:\n yum-config-manager --enable <repo>"

#: ../yumcommands.py:383
msgid "PACKAGE..."
msgstr "CSOMAG..."

#: ../yumcommands.py:390
msgid "Install a package or packages on your system"
msgstr "Csomag(ok) telepítése a rendszerre"

#: ../yumcommands.py:421
msgid "Setting up Install Process"
msgstr "Telepítési folyamat megkezdése"

#: ../yumcommands.py:447 ../yumcommands.py:507
msgid "[PACKAGE...]"
msgstr "[CSOMAG...]"

#: ../yumcommands.py:454
msgid "Update a package or packages on your system"
msgstr "Csomag(ok) frissítése a rendszeren"

#: ../yumcommands.py:483
msgid "Setting up Update Process"
msgstr "Frissítési folyamat megkezdése"

#: ../yumcommands.py:514
msgid "Synchronize installed packages to the latest available versions"
msgstr "Telepített csomagok szinkronizálása a legfrissebb verzióhoz"

#: ../yumcommands.py:543
msgid "Setting up Distribution Synchronization Process"
msgstr "Disztribúció szinkronizálási folyamat megkezdése"

#: ../yumcommands.py:603
msgid "Display details about a package or group of packages"
msgstr "Részletek megjelenítése egy csomagról vagy egy csomagcsoportról"

#: ../yumcommands.py:672
msgid "Installed Packages"
msgstr "Telepített csomagok"

#: ../yumcommands.py:682
msgid "Available Packages"
msgstr "Elérhető csomagok"

#: ../yumcommands.py:687
msgid "Extra Packages"
msgstr "Extra csomagok"

#: ../yumcommands.py:691
msgid "Updated Packages"
msgstr "Frissített csomagok"

#. This only happens in verbose mode
#: ../yumcommands.py:699 ../yumcommands.py:706 ../yumcommands.py:1539
msgid "Obsoleting Packages"
msgstr "Elavult csomagok"

#: ../yumcommands.py:708
msgid "Recently Added Packages"
msgstr "Nemrégiben hozzáadott csomagok"

#: ../yumcommands.py:715
msgid "No matching Packages to list"
msgstr "Nem található csomag"

#: ../yumcommands.py:766
msgid "List a package or groups of packages"
msgstr "Csomag vagy csomagcsoport listázása"

#: ../yumcommands.py:797
msgid "Remove a package or packages from your system"
msgstr "Csomag(ok) eltávolítása a rendszerből"

#: ../yumcommands.py:845
msgid "Setting up Remove Process"
msgstr "Eltávolítási folyamat megkezdése"

#: ../yumcommands.py:906
msgid "Display, or use, the groups information"
msgstr "Megjeleníti, vagy használja a csoportinformációkat"

#: ../yumcommands.py:909
msgid "Setting up Group Process"
msgstr "Csoport folyamat megkezdése"

#: ../yumcommands.py:915
msgid "No Groups on which to run command"
msgstr "Nem található csoport, amin futtatható a parancs"

#: ../yumcommands.py:985
#, python-format
msgid "Invalid groups sub-command, use: %s."
msgstr "Hibás csoport al-parancs, kérem, használja: %s"

#: ../yumcommands.py:992
msgid "There is no installed groups file."
msgstr "Nincs telepített csoportfájl."

#: ../yumcommands.py:994
msgid "You don't have access to the groups DB."
msgstr "Önnek nincs hozzáférési jogosultsága a csoport DB-hez."

#: ../yumcommands.py:1256
msgid "Generate the metadata cache"
msgstr "Metaadat-gyorsítótár generálása"

#: ../yumcommands.py:1282
msgid "Making cache files for all metadata files."
msgstr "Gyorsítótár készítése az összes metaadat fájlhoz."

#: ../yumcommands.py:1283
msgid "This may take a while depending on the speed of this computer"
msgstr "Ez eltarthat egy darabig a számítógép sebességétől függően"

#: ../yumcommands.py:1312
msgid "Metadata Cache Created"
msgstr "Metaadat gyorsítótár létrehozva"

#: ../yumcommands.py:1350
msgid "Remove cached data"
msgstr "Gyorsítótárazott adat eltávolítása"

#: ../yumcommands.py:1417
msgid "Find what package provides the given value"
msgstr "Megtalálja, mely csomag tartalmazza a megadott értéket"

#: ../yumcommands.py:1485
msgid "Check for available package updates"
msgstr "Csomagfrissítések ellenőrzése"

#: ../yumcommands.py:1587
msgid "Search package details for the given string"
msgstr "Csomagrészletek keresése egy megadott szöveg alapján"

#: ../yumcommands.py:1613
msgid "Searching Packages: "
msgstr "Csomagok keresése: "

#: ../yumcommands.py:1666
msgid "Update packages taking obsoletes into account"
msgstr "Csomagok frissítése az elavultakat is számításba véve"

#: ../yumcommands.py:1696
msgid "Setting up Upgrade Process"
msgstr "Frissítési folyamat megkezdése"

#: ../yumcommands.py:1731
msgid "Install a local RPM"
msgstr "Helyi RPM telepítése"

#: ../yumcommands.py:1761
msgid "Setting up Local Package Process"
msgstr "Helyi csomagolási folyamat megkezdése"

#: ../yumcommands.py:1825
msgid "Searching Packages for Dependency:"
msgstr "Csomagfüggőségek keresése:"

#: ../yumcommands.py:1867
msgid "Run an interactive yum shell"
msgstr "Interaktív yum shell futtatása"

#: ../yumcommands.py:1893
msgid "Setting up Yum Shell"
msgstr "Yum Shell indítása"

#: ../yumcommands.py:1936
msgid "List a package's dependencies"
msgstr "Egy csomag függőségeinek listázása"

#: ../yumcommands.py:1963
msgid "Finding dependencies: "
msgstr "Függőségek keresése: "

#: ../yumcommands.py:2005
msgid "Display the configured software repositories"
msgstr "Beállított szoftverforrások megjelenítése"

#: ../yumcommands.py:2094 ../yumcommands.py:2095
msgid "enabled"
msgstr "engedélyezett"

#: ../yumcommands.py:2121 ../yumcommands.py:2122
msgid "disabled"
msgstr "tiltott"

#: ../yumcommands.py:2137
msgid "Repo-id      : "
msgstr "Tároló-azonosító: "

#: ../yumcommands.py:2138
msgid "Repo-name    : "
msgstr "Tároló-név   : "

#: ../yumcommands.py:2141
msgid "Repo-status  : "
msgstr "Tároló-állapot: "

#: ../yumcommands.py:2144
msgid "Repo-revision: "
msgstr "Tároló-vizsgálat:"

#: ../yumcommands.py:2148
msgid "Repo-tags    : "
msgstr "Tároló-címkék : "

#: ../yumcommands.py:2154
msgid "Repo-distro-tags: "
msgstr "Tároló-diszt.-címkék: "

#: ../yumcommands.py:2159
msgid "Repo-updated : "
msgstr "Tároló-frissítve: "

#: ../yumcommands.py:2161
msgid "Repo-pkgs    : "
msgstr "Tároló-csomagok: "

#: ../yumcommands.py:2162
msgid "Repo-size    : "
msgstr "Tároló-méret  : "

#: ../yumcommands.py:2169 ../yumcommands.py:2190
msgid "Repo-baseurl : "
msgstr "Tároló-baseurl: "

#: ../yumcommands.py:2177
msgid "Repo-metalink: "
msgstr "Tároló-metalink: "

#: ../yumcommands.py:2181
msgid "  Updated    : "
msgstr "  Frissítve  : "

#: ../yumcommands.py:2184
msgid "Repo-mirrors : "
msgstr "Tároló-tükrök: "

#: ../yumcommands.py:2199
#, python-format
msgid "Never (last: %s)"
msgstr "Soha (utoljára: %s)"

#: ../yumcommands.py:2201
#, python-format
msgid "Instant (last: %s)"
msgstr "Most (utoljára: %s)"

#: ../yumcommands.py:2204
#, python-format
msgid "%s second(s) (last: %s)"
msgstr "%s másodperc (utoljára: %s)"

#: ../yumcommands.py:2206
msgid "Repo-expire  : "
msgstr "Tároló-lejárat: "

#: ../yumcommands.py:2209
msgid "Repo-exclude : "
msgstr "Tároló-kizárás:"

#: ../yumcommands.py:2213
msgid "Repo-include : "
msgstr "Tároló-tartalmaz: "

#: ../yumcommands.py:2217
msgid "Repo-excluded: "
msgstr "Tároló-kizárt: "

#: ../yumcommands.py:2221
msgid "Repo-filename: "
msgstr "Tároló-fájlnév: "

#. Work out the first (id) and last (enabled/disalbed/count),
#. then chop the middle (name)...
#: ../yumcommands.py:2230 ../yumcommands.py:2259
msgid "repo id"
msgstr "tároló azon"

#: ../yumcommands.py:2247 ../yumcommands.py:2248 ../yumcommands.py:2266
msgid "status"
msgstr "állapot"

#: ../yumcommands.py:2260
msgid "repo name"
msgstr "tároló név"

#: ../yumcommands.py:2332
msgid "Display a helpful usage message"
msgstr "Egy használati tipp mutatása"

#: ../yumcommands.py:2374
#, python-format
msgid "No help available for %s"
msgstr "Nincs súgó az alábbi témakörben: %s"

#: ../yumcommands.py:2379
msgid ""
"\n"
"\n"
"aliases: "
msgstr "\n\nalias-ok: "

#: ../yumcommands.py:2381
msgid ""
"\n"
"\n"
"alias: "
msgstr "\n\nalias: "

#: ../yumcommands.py:2466
msgid "Setting up Reinstall Process"
msgstr "Újratelepítő folyamat indítása"

#: ../yumcommands.py:2478
msgid "reinstall a package"
msgstr "egy csomag újratelepítése"

#: ../yumcommands.py:2541
msgid "Setting up Downgrade Process"
msgstr "Visszaállítási folyamat indítása"

#: ../yumcommands.py:2552
msgid "downgrade a package"
msgstr "egy csomag visszaállítása"

#: ../yumcommands.py:2591
msgid "Display a version for the machine and/or available repos."
msgstr "Verzió és/vagy elérhető tárolók megjelenítése"

#: ../yumcommands.py:2643
msgid " Yum version groups:"
msgstr " Yum verzió csoportok:"

#: ../yumcommands.py:2653
msgid " Group   :"
msgstr " Csoport   :"

#: ../yumcommands.py:2654
msgid " Packages:"
msgstr "Csomagok:"

#: ../yumcommands.py:2683
msgid "Installed:"
msgstr "Telepített:"

#: ../yumcommands.py:2691
msgid "Group-Installed:"
msgstr "Telepített csoport:"

#: ../yumcommands.py:2700
msgid "Available:"
msgstr "Elérhető:"

#: ../yumcommands.py:2709
msgid "Group-Available:"
msgstr "Elérhető csoport:"

#: ../yumcommands.py:2783
msgid "Display, or use, the transaction history"
msgstr "Korábbi tranzakciók használata vagy megjelenítése"

#: ../yumcommands.py:2876 ../yumcommands.py:2880
msgid "Transactions:"
msgstr "Tranzakciók:"

#: ../yumcommands.py:2881
msgid "Begin time  :"
msgstr "Kezdés ideje   :"

#: ../yumcommands.py:2882
msgid "End time    :"
msgstr "Befejezés ideje:"

#: ../yumcommands.py:2883
msgid "Counts      :"
msgstr "Összesen     :"

#: ../yumcommands.py:2884
msgid "  NEVRAC :"
msgstr "  NEVRAC :"

#: ../yumcommands.py:2885
msgid "  NEVRA  :"
msgstr "  NEVRA  :"

#: ../yumcommands.py:2886
msgid "  NA     :"
msgstr "  NA     :"

#: ../yumcommands.py:2887
msgid "  NEVR   :"
msgstr "  NEVR   :"

#: ../yumcommands.py:2888
msgid "  rpm DB :"
msgstr "  rpm DB :"

#: ../yumcommands.py:2889
msgid "  yum DB :"
msgstr "  yum DB :"

#: ../yumcommands.py:2922
#, python-format
msgid "Invalid history sub-command, use: %s."
msgstr "Helytelen alparancs az előzményekhez, használja a következőt: %s"

#: ../yumcommands.py:2929
msgid "You don't have access to the history DB."
msgstr "Ön nem tudja elérni az előzmények adatbázisát."

#: ../yumcommands.py:3036
msgid "Check for problems in the rpmdb"
msgstr "Hibák keresése az rpmdb-ben"

#: ../yumcommands.py:3102
msgid "load a saved transaction from filename"
msgstr "egy elmentett tranzakció betöltése fájlnév alapján"

#: ../yumcommands.py:3119
msgid "No saved transaction file specified."
msgstr "Nincs meghatározva az elmentett tranzakció fájlja."

#: ../yumcommands.py:3123
#, python-format
msgid "loading transaction from %s"
msgstr "tranzakciók betöltések a következőből: %s"

#: ../yumcommands.py:3129
#, python-format
msgid "Transaction loaded from %s with %s members"
msgstr "Tranzakció betöltve innen: %s, %s taggal"

#: ../yumcommands.py:3169
msgid "Simple way to swap packages, isntead of using shell"
msgstr "Egyszerű módja a csomagok cserélésének a shell használata nélkül"

#: ../yumcommands.py:3264
msgid ""
"Treat a repo. as a group of packages, so we can install/remove all of them"
msgstr "Egy tároló csoportként való kezelése, ezáltal mindet tudjuk telepíteni, ill. eltávolítani"

#: ../yumcommands.py:3341
#, python-format
msgid "%d package to update"
msgid_plural "%d packages to update"
msgstr[0] "%d csomag frissítendő"
msgstr[1] "%d csomag frissítendő"

#: ../yumcommands.py:3370
#, python-format
msgid "%d package to remove/reinstall"
msgid_plural "%d packages to remove/reinstall"
msgstr[0] "%d csomag eltávolítandó/újratelepítendő"
msgstr[1] "%d csomag eltávolítandó/újratelepítendő"

#: ../yumcommands.py:3413
#, python-format
msgid "%d package to remove/sync"
msgid_plural "%d packages to remove/sync"
msgstr[0] "%d csomag eltávolítandó/szinkronizálandó"
msgstr[1] "%d csomag eltávolítandó/szinkronizálandó"

#: ../yumcommands.py:3417
#, python-format
msgid "Not a valid sub-command of %s"
msgstr "Nincs érvényes al-parancs a következőhöz: %s"

#. This is mainly for PackageSackError from rpmdb.
#: ../yummain.py:84
#, python-format
msgid " Yum checks failed: %s"
msgstr " Yum ellenőrzés meghiúsult: %s"

#: ../yummain.py:98
msgid "No read/execute access in current directory, moving to /"
msgstr "Nincs olvasási/végrehajtási jogosultság a jelenlegi könyvtárban, továbblépés ide: /"

#: ../yummain.py:106
msgid "No getcwd() access in current directory, moving to /"
msgstr "Nincs getcwd() jogosultság a jelenlegi könyvtárban, továbblépés ide: /"

#. Depsolve stage
#: ../yummain.py:164
msgid "Resolving Dependencies"
msgstr "Függőségek megállapítása"

#: ../yummain.py:227 ../yummain.py:235
#, python-format
msgid ""
"Your transaction was saved, rerun it with:\n"
" yum load-transaction %s"
msgstr "Az ön tranzakciója mentve, újra futtathatja a következővel:\n yum load-transaction %s"

#: ../yummain.py:312
msgid ""
"\n"
"\n"
"Exiting on user cancel."
msgstr "\n\nKilépés felhasználói megszakítás miatt."

#: ../yum/depsolve.py:127
msgid "doTsSetup() will go away in a future version of Yum.\n"
msgstr "A doTsSetup() már nem lesz elérhető a későbbi Yum verziókban.\n"

#: ../yum/depsolve.py:143
msgid "Setting up TransactionSets before config class is up"
msgstr "Tranzakciók indítása a konfigurációs osztályok felállása előtt"

#: ../yum/depsolve.py:200
#, python-format
msgid "Invalid tsflag in config file: %s"
msgstr "Hibás tsflag a következő konfigurációs fájlban: %s"

#: ../yum/depsolve.py:218
#, python-format
msgid "Searching pkgSack for dep: %s"
msgstr "Keresés a pkgSack-ben függőség után: %s"

#: ../yum/depsolve.py:269
#, python-format
msgid "Member: %s"
msgstr "Tag: %s"

#: ../yum/depsolve.py:283 ../yum/depsolve.py:937
#, python-format
msgid "%s converted to install"
msgstr "A(z) %s át lett konvertálva telepítéshez"

#: ../yum/depsolve.py:295
#, python-format
msgid "Adding Package %s in mode %s"
msgstr "A(z) %s nevű csomag hozzáadása %s módban"

#: ../yum/depsolve.py:311
#, python-format
msgid "Removing Package %s"
msgstr "A következő csomag eltávolítása: %s"

#: ../yum/depsolve.py:333
#, python-format
msgid "%s requires: %s"
msgstr "A(z) %s megköveteli a következőket: %s"

#: ../yum/depsolve.py:374
#, python-format
msgid "%s requires %s"
msgstr "A(z) %s megköveteli a következőt: %s"

#: ../yum/depsolve.py:401
msgid "Needed Require has already been looked up, cheating"
msgstr "A szükséges dolognak már utánanéztek, lecsaljuk"

#: ../yum/depsolve.py:411
#, python-format
msgid "Needed Require is not a package name. Looking up: %s"
msgstr "A megkövetelt dolog nem csomag, ezért utánanézünk: %s"

#: ../yum/depsolve.py:419
#, python-format
msgid "Potential Provider: %s"
msgstr "Lehetséges Szolgáltató: %s"

#: ../yum/depsolve.py:442
#, python-format
msgid "Mode is %s for provider of %s: %s"
msgstr "A mód %s a(z) %s kiszolgáló felé: %s"

#: ../yum/depsolve.py:446
#, python-format
msgid "Mode for pkg providing %s: %s"
msgstr "A(z) %s nevű csomag szolgáltatásának módja: %s"

#. the thing it needs is being updated or obsoleted away
#. try to update the requiring package in hopes that all this problem goes
#. away :(
#: ../yum/depsolve.py:451 ../yum/depsolve.py:486
#, python-format
msgid "Trying to update %s to resolve dep"
msgstr "Kisérlet a(z) %s frissítésére a függőségek feloldásához"

#: ../yum/depsolve.py:480
#, python-format
msgid "No update paths found for %s. Failure!"
msgstr "Nem található frissítési útvonal a következőhöz: %s."

#: ../yum/depsolve.py:491
#, python-format
msgid "No update paths found for %s. Failure due to requirement: %s!"
msgstr "Nem található frissítési útvonal a következőhöz: %s. A hiba ezen követelményből ered: %s!"

#: ../yum/depsolve.py:507
#, python-format
msgid "Update for %s. Doesn't fix requirement: %s!"
msgstr "Frissítés a következőhöz: %s. Nem javítja a követelményt: %s!"

#: ../yum/depsolve.py:514
#, python-format
msgid "TSINFO: %s package requiring %s marked as erase"
msgstr "TSINFO: A(z) %s nevű csomaghoz szükséges a(z) %s törlése"

#: ../yum/depsolve.py:527
#, python-format
msgid "TSINFO: Obsoleting %s with %s to resolve dep."
msgstr "TSINFO: %s lecserélése függőség miatt a következőre: %s."

#: ../yum/depsolve.py:530
#, python-format
msgid "TSINFO: Updating %s to resolve dep."
msgstr "TSINFO: %s frissítése függőség miatt."

#: ../yum/depsolve.py:538
#, python-format
msgid "Cannot find an update path for dep for: %s"
msgstr "Nem található frissítési útvonal a következőhöz: %s"

#: ../yum/depsolve.py:569
#, python-format
msgid "Quick matched %s to require for %s"
msgstr "A(z) %s követelménye %s (gyors találattal)"

#. is it already installed?
#: ../yum/depsolve.py:611
#, python-format
msgid "%s is in providing packages but it is already installed, removing."
msgstr "%s az ellátandó csomagoknál található, bár telepítve van, ezért eltávolítjuk."

#: ../yum/depsolve.py:627
#, python-format
msgid "Potential resolving package %s has newer instance in ts."
msgstr "A lehetséges %s nevű csomag már újabb verzióval szerepel a tranzakcióban."

#: ../yum/depsolve.py:638
#, python-format
msgid "Potential resolving package %s has newer instance installed."
msgstr "A lehetséges %s nevű csomagból már újabb verzió van telepítve."

#: ../yum/depsolve.py:656
#, python-format
msgid "%s already in ts, skipping this one"
msgstr "A(z) %s már szerepel a tranzakcióban, kihagyás"

#: ../yum/depsolve.py:705
#, python-format
msgid "TSINFO: Marking %s as update for %s"
msgstr "TSINFO: %s bejelölése a(z) %s frissítéseként"

#: ../yum/depsolve.py:714
#, python-format
msgid "TSINFO: Marking %s as install for %s"
msgstr "TSINFO: A(z) %s telepítésre való bejelölése a következőhöz: %s"

#: ../yum/depsolve.py:849 ../yum/depsolve.py:967
msgid "Success - empty transaction"
msgstr "Siker - üres tranzakció"

#: ../yum/depsolve.py:889 ../yum/depsolve.py:927
msgid "Restarting Loop"
msgstr "Folyamat újraindítása"

#: ../yum/depsolve.py:947
msgid "Dependency Process ending"
msgstr "Függőségkezelő folyamat végetért"

#: ../yum/depsolve.py:969
msgid "Success - deps resolved"
msgstr "Siker - függőségek megoldva"

#: ../yum/depsolve.py:993
#, python-format
msgid "Checking deps for %s"
msgstr "Függőségek keresése a következőre: %s"

#: ../yum/depsolve.py:1082
#, python-format
msgid "looking for %s as a requirement of %s"
msgstr "A(z) %s keresése, mint a(z) %s szükséglete."

#: ../yum/depsolve.py:1349
#, python-format
msgid "Running compare_providers() for %s"
msgstr "A compare_providers() parancs futtatása a következőhöz: %s"

#: ../yum/depsolve.py:1376 ../yum/depsolve.py:1382
#, python-format
msgid "better arch in po %s"
msgstr "Jobb architektúra a következőben: %s"

#: ../yum/depsolve.py:1496
#, python-format
msgid "%s obsoletes %s"
msgstr "A(z) %s már újabb, mint a(z) %s"

#: ../yum/depsolve.py:1508
#, python-format
msgid ""
"archdist compared %s to %s on %s\n"
"  Winner: %s"
msgstr "archdist osszehasonlítva %s a következőhöz: %s a következőn: %s\n  Nyertes: %s"

#: ../yum/depsolve.py:1516
#, python-format
msgid "common sourcerpm %s and %s"
msgstr "gyakori forrásrpm %s és %s"

#: ../yum/depsolve.py:1520
#, python-format
msgid "base package %s is installed for %s"
msgstr "legjobb %s nevű csomag telepítve van a következőhöz: %s"

#: ../yum/depsolve.py:1526
#, python-format
msgid "common prefix of %s between %s and %s"
msgstr "gyakori előtag a következőre: %s, %s és %s között"

#: ../yum/depsolve.py:1543
#, python-format
msgid "provides vercmp: %s"
msgstr "provides vercmp: %s"

#: ../yum/depsolve.py:1547 ../yum/depsolve.py:1581
#, python-format
msgid " Winner: %s"
msgstr " Nyertes: %s"

#: ../yum/depsolve.py:1577
#, python-format
msgid "requires minimal: %d"
msgstr "minimum szükséges: %d"

#: ../yum/depsolve.py:1586
#, python-format
msgid " Loser(with %d): %s"
msgstr " Vesztes(ezzel: %d): %s"

#: ../yum/depsolve.py:1602
#, python-format
msgid "Best Order: %s"
msgstr "Legjobb sorrend: %s"

#: ../yum/__init__.py:274
msgid "doConfigSetup() will go away in a future version of Yum.\n"
msgstr "A doConfigSetup() már nem lesz elérhető a későbbi Yum verziókban.\n"

#: ../yum/__init__.py:553
#, python-format
msgid "Skipping unreadable repository %s"
msgstr "Nem olvasható tároló kihagyása: %s"

#: ../yum/__init__.py:572
#, python-format
msgid "Repository %r: Error parsing config: %s"
msgstr "%r tároló: Hiba a konfigurációs fájl elemzése során: %s"

#: ../yum/__init__.py:578
#, python-format
msgid "Repository %r is missing name in configuration, using id"
msgstr "A %r tárolónak hiányzik a neve a konfigurációban, ezért azonosítót használunk."

#: ../yum/__init__.py:618
msgid "plugins already initialised"
msgstr "a bővítmények már betöltődtek"

#: ../yum/__init__.py:627
msgid "doRpmDBSetup() will go away in a future version of Yum.\n"
msgstr "A doRpmDBSetup() már nem lesz elérhető a későbbi Yum verziókban.\n"

#: ../yum/__init__.py:638
msgid "Reading Local RPMDB"
msgstr "Helyi RPMDB beolvasása"

#: ../yum/__init__.py:668
msgid "doRepoSetup() will go away in a future version of Yum.\n"
msgstr "A doRepoSetup() már nem lesz elérhető a későbbi Yum verziókban.\n"

#: ../yum/__init__.py:722
msgid "doSackSetup() will go away in a future version of Yum.\n"
msgstr "A doSackSetup() már nem lesz elérhető a későbbi Yum verziókban.\n"

#: ../yum/__init__.py:752
msgid "Setting up Package Sacks"
msgstr "Tárolók csomagjainak előkészítése"

#: ../yum/__init__.py:797
#, python-format
msgid "repo object for repo %s lacks a _resetSack method\n"
msgstr "A következő tárolóobjektumból hiányzik a _resetSack metódus: %s\n"

#: ../yum/__init__.py:798
msgid "therefore this repo cannot be reset.\n"
msgstr "ezért ezt a tárolót nem lehet visszaállítani.\n"

#: ../yum/__init__.py:806
msgid "doUpdateSetup() will go away in a future version of Yum.\n"
msgstr "A doUpdateSetup() már nem lesz elérhető a későbbi Yum verziókban.\n"

#: ../yum/__init__.py:818
msgid "Building updates object"
msgstr "Frissítési objektum létrehozása"

#: ../yum/__init__.py:862
msgid "doGroupSetup() will go away in a future version of Yum.\n"
msgstr "A doGroupSetup() már nem lesz elérhető a későbbi Yum verziókban.\n"

#: ../yum/__init__.py:887
msgid "Getting group metadata"
msgstr "Csoport metaadatok beszerzése"

#: ../yum/__init__.py:915
#, python-format
msgid "Adding group file from repository: %s"
msgstr "Csoportfájl hozzáadása a következő tárolóból: %s"

#: ../yum/__init__.py:918
#, python-format
msgid "Failed to retrieve group file for repository: %s"
msgstr "Nem sikerült megszerezni a csoport fájlt a tárolóhoz: %s"

#: ../yum/__init__.py:924
#, python-format
msgid "Failed to add groups file for repository: %s - %s"
msgstr "Sikertelen a csoportfájl hozzáadása a következő tárolónál: %s - %s"

#: ../yum/__init__.py:930
msgid "No Groups Available in any repository"
msgstr "Nincsenek elérhető csoportok egy tárolóban sem"

#: ../yum/__init__.py:945
msgid "Getting pkgtags metadata"
msgstr "pkgtags metaadatok beszerzése"

#: ../yum/__init__.py:955
#, python-format
msgid "Adding tags from repository: %s"
msgstr "Címkék hozzáadása a következő tárolóból: %s"

#: ../yum/__init__.py:966
#, python-format
msgid "Failed to add Pkg Tags for repository: %s - %s"
msgstr "Sikertelen a címkék hozzáadása a következő tárolóból:%s - %s"

#: ../yum/__init__.py:1059
msgid "Importing additional filelist information"
msgstr "További fájllista információk importálása"

#: ../yum/__init__.py:1077
#, python-format
msgid "The program %s%s%s is found in the yum-utils package."
msgstr "A következő program megtalálható a yum-utils csomagban: %s%s%s"

#: ../yum/__init__.py:1094
msgid ""
"There are unfinished transactions remaining. You might consider running yum-"
"complete-transaction first to finish them."
msgstr "Félbehagyott tranzakciók találhatóak. Ha be kívánja azokat fejezni, használja a yum-complete-transaction parancsot."

#: ../yum/__init__.py:1111
msgid "--> Finding unneeded leftover dependencies"
msgstr "--> Szükségtelen függőségek keresése"

#: ../yum/__init__.py:1169
#, python-format
msgid "Protected multilib versions: %s != %s"
msgstr "Védett multilib verzió: %s != %s"

#. People are confused about protected mutilib ... so give
#. them a nicer message.
#: ../yum/__init__.py:1173
#, python-format
msgid ""
" Multilib version problems found. This often means that the root\n"
"cause is something else and multilib version checking is just\n"
"pointing out that there is a problem. Eg.:\n"
"\n"
"  1. You have an upgrade for %(name)s which is missing some\n"
"     dependency that another package requires. Yum is trying to\n"
"     solve this by installing an older version of %(name)s of the\n"
"     different architecture. If you exclude the bad architecture\n"
"     yum will tell you what the root cause is (which package\n"
"     requires what). You can try redoing the upgrade with\n"
"     --exclude %(name)s.otherarch ... this should give you an error\n"
"     message showing the root cause of the problem.\n"
"\n"
"  2. You have multiple architectures of %(name)s installed, but\n"
"     yum can only see an upgrade for one of those arcitectures.\n"
"     If you don't want/need both architectures anymore then you\n"
"     can remove the one with the missing update and everything\n"
"     will work.\n"
"\n"
"  3. You have duplicate versions of %(name)s installed already.\n"
"     You can use \"yum check\" to get yum show these errors.\n"
"\n"
"...you can also use --setopt=protected_multilib=false to remove\n"
"this checking, however this is almost never the correct thing to\n"
"do as something else is very likely to go wrong (often causing\n"
"much more problems).\n"
"\n"
msgstr "Multilib verzióból adódó hiba található. Ez gyakran azt jeleneti, hogy\na probléma gyökere máshol van, a multilib verzió ellenőrzés csak\nrámutat hiba létezésére. Pl.:\n\n1. Egy frissítés a(z) %(name)s csomaghoz nem talál egy függőséget,\namit egy másik megkövetel. A yum ezt a(z) %(name)s csomag régebbi\nváltozatának a telepítésével próbálja orvosolni az eltérő architektúrához.\nHa ön kizárja a rossz architektúrát, a yum meg fogja mondani a hiba\ngyökerét (mely csomag mit követel meg). Újra megpróbálhatja a telepítést\naz --exclude %(name)s.otherarch ... paraméterekkel. Ezzel egy\nhibaüzenethez jutunk el, ami rámutat a hiba gyökerére.\n\n2. Önnek több %(name)s nevű csomagja telepítve van különböző\narchitektúrákkal, viszont a yum csak az egyiknek a frissítését látja.\nHa önnek már nincs szüksége mindegyik architektúrára, akkor\neltávolíthatja az egyiket, melynél hiányoznak a frissítések, így\nminden működni fog újra.\n\n3. Önnek több verzióval van telepítve a(z) %(name)s nevű csomagja.\nHasználhatja a \"yum check\" parancsot ezen hibák megjelenítésére.\n\n...továbbá használhatja a --setopt=protected_multilib=false kapcsolót\nezen ellenőrzés kihagyásához, habár ez szinte sosem javítja meg a hibás\nműködést, inkább csak tovább ront a helyzeten.\n\n"

#: ../yum/__init__.py:1257
#, python-format
msgid "Trying to remove \"%s\", which is protected"
msgstr "Próbálkozás a(z) \"%s\" eltávolításával, ami védett"

#: ../yum/__init__.py:1378
msgid ""
"\n"
"Packages skipped because of dependency problems:"
msgstr "\nAz alábbi csomagok ki lettek hagyva függőségi problémák miatt:"

#: ../yum/__init__.py:1382
#, python-format
msgid "    %s from %s"
msgstr "    %s a következőből: %s"

#. FIXME: _N()
#: ../yum/__init__.py:1556
#, python-format
msgid "** Found %d pre-existing rpmdb problem(s), 'yum check' output follows:"
msgstr "** %d már meglévő rpmdb probléma található, 'yum check' kimenet pedig a következő:"

#: ../yum/__init__.py:1560
msgid "Warning: RPMDB altered outside of yum."
msgstr "Figyelem: RPMDB a yumon kívülről lett megváltoztatva."

#: ../yum/__init__.py:1572
msgid "missing requires"
msgstr "követelmények hiányoznak"

#: ../yum/__init__.py:1573
msgid "installed conflict"
msgstr "konfliktusokat teremtett"

#: ../yum/__init__.py:1709
msgid ""
"Warning: scriptlet or other non-fatal errors occurred during transaction."
msgstr "Figyelem: scriptlet vagy egyéb nem végzetes hiba adódott a tranzakció során."

#: ../yum/__init__.py:1719
msgid "Transaction couldn't start:"
msgstr "A tranzakció nem indítható:"

#. should this be 'to_unicoded'?
#: ../yum/__init__.py:1722
msgid "Could not run transaction."
msgstr "Tranzakció futtatása meghiúsult."

#: ../yum/__init__.py:1736
#, python-format
msgid "Failed to remove transaction file %s"
msgstr "A következő tranzakció-fájl eltávolítása meghiúsult: %s"

#. maybe a file log here, too
#. but raising an exception is not going to do any good
#: ../yum/__init__.py:1792
#, python-format
msgid "%s was supposed to be installed but is not!"
msgstr "A(z) %s nevű csomagnak már telepítve kellett volna lennie, habár nincs."

#. maybe a file log here, too
#. but raising an exception is not going to do any good
#. Note: This actually triggers atm. because we can't
#. always find the erased txmbr to set it when
#. we should.
#: ../yum/__init__.py:1869
#, python-format
msgid "%s was supposed to be removed but is not!"
msgstr "A(z) %s nevű csomagok már el kellett volna távolítani, habár ez még nem történt meg."

#. Another copy seems to be running.
#: ../yum/__init__.py:2004
#, python-format
msgid "Existing lock %s: another copy is running as pid %s."
msgstr "Már le van zárva %s: egy másik példány fut a következő pid-vel: %s"

#. Whoa. What the heck happened?
#: ../yum/__init__.py:2053
#, python-format
msgid "Could not create lock at %s: %s "
msgstr "Nem zárható le a következő: %s: %s "

#: ../yum/__init__.py:2065
#, python-format
msgid "Could not open lock %s: %s"
msgstr "A zárolás feloldása nem lehetséges: %s: %s"

#. The pid doesn't exist
#. Whoa. What the heck happened?
#: ../yum/__init__.py:2082
#, python-format
msgid "Unable to check if PID %s is active"
msgstr "Nem lehetséges az ellenőrzés, ha a %s PID aktív"

#: ../yum/__init__.py:2132
#, python-format
msgid ""
"Package does not match intended download. Suggestion: run yum "
"--enablerepo=%s clean metadata"
msgstr "A csomag nem egyezik a várt letöltéssel. Ajánlott futtatni az alábbi parancsot:\nyum --enablerepo=%s clean metadata"

#: ../yum/__init__.py:2155
msgid "Could not perform checksum"
msgstr "Ellenőrzőösszeg végrehajtása nem lehetséges"

#: ../yum/__init__.py:2158
msgid "Package does not match checksum"
msgstr "A csomag nem egyezik az ellenőrzőösszeggel"

#: ../yum/__init__.py:2222
#, python-format
msgid "package fails checksum but caching is enabled for %s"
msgstr "A csomag nem egyezik az ellenőrzőösszeggel, de a gyorsítótárazás engedélyezett a következőhöz: %s"

#: ../yum/__init__.py:2225 ../yum/__init__.py:2268
#, python-format
msgid "using local copy of %s"
msgstr "helyi másolat használata a következőhöz: %s"

#. caller handles errors
#: ../yum/__init__.py:2342
msgid "exiting because --downloadonly specified"
msgstr "kilépés a --downloadonly kapcsoló miatt"

#: ../yum/__init__.py:2371
msgid "Header is not complete."
msgstr "A fejléc nem teljes."

#: ../yum/__init__.py:2411
#, python-format
msgid ""
"Header not in local cache and caching-only mode enabled. Cannot download %s"
msgstr "A fejléc nem található meg a helyi gyorsítótárban és a \"csak gyorsítótárazás\" mód be van kapcsolva. Nem tölthető le a következő: %s"

#: ../yum/__init__.py:2471
#, python-format
msgid "Public key for %s is not installed"
msgstr "A publikus kulcs nincs telepítve a következőhöz: %s"

#: ../yum/__init__.py:2475
#, python-format
msgid "Problem opening package %s"
msgstr "Hiba a következő csomag megnyitásánál: %s"

#: ../yum/__init__.py:2483
#, python-format
msgid "Public key for %s is not trusted"
msgstr "A publikus kulcs nem megbízható a következőhöz: %s"

#: ../yum/__init__.py:2487
#, python-format
msgid "Package %s is not signed"
msgstr "A következő csomag nincs aláírva: %s"

#: ../yum/__init__.py:2529
#, python-format
msgid "Cannot remove %s"
msgstr "Nem távolítható el: %s"

#: ../yum/__init__.py:2533
#, python-format
msgid "%s removed"
msgstr "%s eltávolítva"

#: ../yum/__init__.py:2594
#, python-format
msgid "Cannot remove %s file %s"
msgstr "Nem távolítható el a %s nevű fájl: %s"

#: ../yum/__init__.py:2598
#, python-format
msgid "%s file %s removed"
msgstr "%s fájl %s eltávolította"

#: ../yum/__init__.py:2600
#, python-format
msgid "%d %s file removed"
msgid_plural "%d %s files removed"
msgstr[0] "%d %s fájl eltávolítva"
msgstr[1] "%d %s fájl eltávolítva"

#: ../yum/__init__.py:2712
#, python-format
msgid "More than one identical match in sack for %s"
msgstr "Több, mint egy azonos találat a tömbben a következőre: %s"

#: ../yum/__init__.py:2718
#, python-format
msgid "Nothing matches %s.%s %s:%s-%s from update"
msgstr "Semmi sem egyezik a következővel: %s.%s %s:%s-%s a frissítésből"

#: ../yum/__init__.py:3096
msgid ""
"searchPackages() will go away in a future version of Yum."
"                      Use searchGenerator() instead. \n"
msgstr "A searchPackages() parancs már nem lesz elérhető a későbbi Yum verziókban.\nKérjük, inkább használja a searchGenerator() parancsot helyette.\n"

#: ../yum/__init__.py:3149
#, python-format
msgid "Searching %d package"
msgid_plural "Searching %d packages"
msgstr[0] "%d csomag keresése"
msgstr[1] "%d csomag keresése"

#: ../yum/__init__.py:3153
#, python-format
msgid "searching package %s"
msgstr "A következő csomag keresése: %s"

#: ../yum/__init__.py:3165
msgid "searching in file entries"
msgstr "Keresés fájlbejegyzésekben"

#: ../yum/__init__.py:3172
msgid "searching in provides entries"
msgstr "Keresés ellátási bejegyzésekben"

#: ../yum/__init__.py:3369
msgid "No group data available for configured repositories"
msgstr "Nincsenek elérhető csoportadatok a beállított tárolókban"

#: ../yum/__init__.py:3466 ../yum/__init__.py:3500 ../yum/__init__.py:3576
#: ../yum/__init__.py:3582 ../yum/__init__.py:3719 ../yum/__init__.py:3723
#: ../yum/__init__.py:4298
#, python-format
msgid "No Group named %s exists"
msgstr "Nincs %s nevű csoport"

#: ../yum/__init__.py:3512 ../yum/__init__.py:3740
#, python-format
msgid "package %s was not marked in group %s"
msgstr "A(z) %s nevű csomag nem lett bejelölve a következő csoportban: %s"

#. (upgrade and igroup_data[pkg] == 'available')):
#: ../yum/__init__.py:3622
#, python-format
msgid "Skipping package %s from group %s"
msgstr "%s csomag átugrása a(z) %s csoportból"

#: ../yum/__init__.py:3628
#, python-format
msgid "Adding package %s from group %s"
msgstr "A(z) %s nevű csomag hozzáadás a következő csoportból: %s"

#: ../yum/__init__.py:3649
#, python-format
msgid "No package named %s available to be installed"
msgstr "Nincs telepíthető csomag %s néven."

#: ../yum/__init__.py:3702
#, python-format
msgid "Warning: Group %s does not have any packages to install."
msgstr "Figyelem: A(z) %s nevű csoport nem tartalmaz telepíthető csomagokat."

#: ../yum/__init__.py:3704
#, python-format
msgid "Group %s does have %u conditional packages, which may get installed."
msgstr "A(z) %s csoportnak %u feltételes csomagja van, amelyek lehet, hogy telepítésre kerülnek."

#: ../yum/__init__.py:3794
#, python-format
msgid "Skipping group %s from environment %s"
msgstr "A(z) %s csoport kihagyása a következő környezetből: %s"

#. This can happen due to excludes after .up has
#. happened.
#: ../yum/__init__.py:3858
#, python-format
msgid "Package tuple %s could not be found in packagesack"
msgstr "A csomagtömbben nem található leíró a következőre: %s"

#: ../yum/__init__.py:3886
#, python-format
msgid "Package tuple %s could not be found in rpmdb"
msgstr "Az rpmdb nem tartalmaz csomagleírót a következőre: %s"

#: ../yum/__init__.py:3949 ../yum/__init__.py:4012
#, python-format
msgid "Invalid version flag from: %s"
msgstr "Érvénytelen verzió: %s"

#: ../yum/__init__.py:3973 ../yum/__init__.py:3979 ../yum/__init__.py:4036
#: ../yum/__init__.py:4042
#, python-format
msgid "No Package found for %s"
msgstr "Nem található csomag a következőre: %s"

#: ../yum/__init__.py:4245 ../yum/__init__.py:4274
#, python-format
msgid "Warning: Environment Group %s does not exist."
msgstr "Figyelem: A(z) %s Környezeti Csoport nem létezik."

#: ../yum/__init__.py:4397
#, python-format
msgid "Package: %s - can't co-install with %s"
msgstr "Csomag: %s - nem használható a co-install telepítése a következővel: %s"

#: ../yum/__init__.py:4437
msgid "Package Object was not a package object instance"
msgstr "A Csomag Objektum nem egy csomag-objektum példány"

#: ../yum/__init__.py:4441
msgid "Nothing specified to install"
msgstr "Semmi sem lett megadva telepítésnek"

#: ../yum/__init__.py:4465 ../yum/__init__.py:5410
#, python-format
msgid "Checking for virtual provide or file-provide for %s"
msgstr "Virtuális szolgáltatása vagy fájlszolgáltatás ellenőrzése a következőre: %s"

#: ../yum/__init__.py:4542
#, python-format
msgid "Package %s installed and not available"
msgstr "A(z) %s nevű csomag telepítve, bár nem érhető el"

#: ../yum/__init__.py:4545
msgid "No package(s) available to install"
msgstr "Nincsenek elérhető csomagok telepítésre"

#: ../yum/__init__.py:4557
#, python-format
msgid "Package: %s  - already in transaction set"
msgstr "Csomag: %s  - már szerepel a tranzakcióban"

#: ../yum/__init__.py:4589
#, python-format
msgid "Package %s is obsoleted by %s which is already installed"
msgstr "A csomag %s cserélve lett a következőre: %s, ami már telepítve van"

#: ../yum/__init__.py:4594
#, python-format
msgid ""
"Package %s is obsoleted by %s, but obsoleting package does not provide for "
"requirements"
msgstr "A(z) %s nevű csomag már elavult a következő miatt: %s, bár ez nem teljesíti a követelményeket"

#: ../yum/__init__.py:4597
#, python-format
msgid "Package %s is obsoleted by %s, trying to install %s instead"
msgstr "A(z) %s nevű csomag már elavult a következő miatt: %s, próbálkozás inkább a(z) %s telepítésével"

#: ../yum/__init__.py:4605
#, python-format
msgid "Package %s already installed and latest version"
msgstr "A(z) %s nevű csomag már a legfrissebb verzióval rendelkezik"

#: ../yum/__init__.py:4619
#, python-format
msgid "Package matching %s already installed. Checking for update."
msgstr "A csomag, mely megegyezik a következővel: %s, már telepítve van. Frissítések keresése."

#. update everything (the easy case)
#: ../yum/__init__.py:4751
msgid "Updating Everything"
msgstr "Minden frissítése"

#: ../yum/__init__.py:4775 ../yum/__init__.py:4930 ../yum/__init__.py:4975
#: ../yum/__init__.py:5011
#, python-format
msgid "Not Updating Package that is already obsoleted: %s.%s %s:%s-%s"
msgstr "Azon csomagok nem lesznek frissítve, melyek már elavultak: %s.%s %s:%s-%s"

#: ../yum/__init__.py:4830 ../yum/__init__.py:5072
#, python-format
msgid "%s"
msgstr "%s"

#: ../yum/__init__.py:4854 ../yum/__init__.py:5080 ../yum/__init__.py:5416
#, python-format
msgid "No Match for argument: %s"
msgstr "Nincs találat a következő argumentumra: %s"

#: ../yum/__init__.py:4872
#, python-format
msgid "No package matched to upgrade: %s"
msgstr "Nincs frissítendő csomag az alábbira: %s"

#: ../yum/__init__.py:4919
#, python-format
msgid "Package is already obsoleted: %s.%s %s:%s-%s"
msgstr "A következő csomag már elavult: %s.%s %s:%s-%s"

#: ../yum/__init__.py:4970
#, python-format
msgid "Not Updating Package that is obsoleted: %s"
msgstr "A következő elavult csomag nem lesz frissítve: %s"

#: ../yum/__init__.py:4979 ../yum/__init__.py:5015
#, python-format
msgid "Not Updating Package that is already updated: %s.%s %s:%s-%s"
msgstr "Azon csomagok nem lesznek frissítve, melyek már frissítve vannak: %s.%s %s:%s-%s"

#: ../yum/__init__.py:5093
#, python-format
msgid "No package matched to remove: %s"
msgstr "Nincs eltávolítandó csomag az alábbira: %s"

#: ../yum/__init__.py:5099
#, python-format
msgid "Skipping the running kernel: %s"
msgstr "A futó kernel kihagyása: %s"

#: ../yum/__init__.py:5105
#, python-format
msgid "Removing %s from the transaction"
msgstr "%s eltávolítása a tranzakcióból"

#: ../yum/__init__.py:5142
#, python-format
msgid "Cannot open: %s. Skipping."
msgstr "Nem nyitható meg a következő: %s. Kihagyás."

#: ../yum/__init__.py:5145 ../yum/__init__.py:5262 ../yum/__init__.py:5347
#, python-format
msgid "Examining %s: %s"
msgstr "%s felülvizsgálata: %s"

#: ../yum/__init__.py:5149
#, python-format
msgid "Cannot localinstall deltarpm: %s. Skipping."
msgstr "Nem lehet helyileg telepíteni a következő deltarpm fájlt: %s. Kihagyás."

#: ../yum/__init__.py:5158 ../yum/__init__.py:5265 ../yum/__init__.py:5350
#, python-format
msgid ""
"Cannot add package %s to transaction. Not a compatible architecture: %s"
msgstr "Nem lehet a(z) %s csomagot hozzáadni a tranzakcióhoz. Nincs kompatibilis architektúra: %s"

#: ../yum/__init__.py:5164
#, python-format
msgid "Cannot install package %s. It is obsoleted by installed package %s"
msgstr "Nem lehet telepíteni a következő csomagot: %s. Ez már elavultnak számít a ezen telepített csomag miatt: %s"

#: ../yum/__init__.py:5172
#, python-format
msgid ""
"Package %s not installed, cannot update it. Run yum install to install it "
"instead."
msgstr "A(z) %s nevű csomag már telepítve van, frissítése nem lehetséges. Kérjük, használja ehelyett a yum install parancsot a telepítéséhez."

#: ../yum/__init__.py:5191 ../yum/__init__.py:5198
#, python-format
msgid ""
"Package %s.%s not installed, cannot update it. Run yum install to install it"
" instead."
msgstr "A következő csomag nincs telepítve: %s.%s, ezért nem lehet frissíteni. Futtassa a \"yum install\" parancsot a telepítéshez."

#: ../yum/__init__.py:5207 ../yum/__init__.py:5270 ../yum/__init__.py:5355
#, python-format
msgid "Excluding %s"
msgstr "%s kizárása"

#: ../yum/__init__.py:5212
#, python-format
msgid "Marking %s to be installed"
msgstr "%s megjelölése telepítésre"

#: ../yum/__init__.py:5218
#, python-format
msgid "Marking %s as an update to %s"
msgstr "%s megjelölése, mint %s frissítése"

#: ../yum/__init__.py:5225
#, python-format
msgid "%s: does not update installed package."
msgstr "%s: nem frissíti a telepített csomagot."

#: ../yum/__init__.py:5259 ../yum/__init__.py:5344
#, python-format
msgid "Cannot open file: %s. Skipping."
msgstr "Nem nyitható meg a következő fájl: %s. Kihagyás."

#: ../yum/__init__.py:5299
msgid "Problem in reinstall: no package matched to remove"
msgstr "Probléma az újratelepítésnél: egy csomag sem eltávolítandó"

#: ../yum/__init__.py:5325
#, python-format
msgid "Problem in reinstall: no package %s matched to install"
msgstr "Probléma az újratelepítésnél: nem található %s nevű csomag telepítéshez"

#: ../yum/__init__.py:5438
msgid "No package(s) available to downgrade"
msgstr "Nincsenek elérhető csomagok visszaállításhoz"

#: ../yum/__init__.py:5446
#, python-format
msgid "Package %s is allowed multiple installs, skipping"
msgstr "A(z) %s csomag többször is telepíthető, kihagyás"

#: ../yum/__init__.py:5496
#, python-format
msgid "No Match for available package: %s"
msgstr "Nem található csomag a következőre: %s"

#: ../yum/__init__.py:5506
#, python-format
msgid "Only Upgrade available on package: %s"
msgstr "Csak frissítés érhető el a következő csomagnál: %s"

#: ../yum/__init__.py:5619 ../yum/__init__.py:5686
#, python-format
msgid "Failed to downgrade: %s"
msgstr "Visszaállítás meghiúsult: %s"

#: ../yum/__init__.py:5636 ../yum/__init__.py:5692
#, python-format
msgid "Failed to upgrade: %s"
msgstr "Sikertelen frissítés: %s"

#: ../yum/__init__.py:5725
#, python-format
msgid "Retrieving key from %s"
msgstr "GPG kulcs beszerzése a következőből: %s"

#: ../yum/__init__.py:5743
msgid "GPG key retrieval failed: "
msgstr "GPG kulcs beszerzés meghiúsult: "

#. if we decide we want to check, even though the sig failed
#. here is where we would do that
#: ../yum/__init__.py:5766
#, python-format
msgid "GPG key signature on key %s does not match CA Key for repo: %s"
msgstr "A(z) %s GPG kulcs aláírása nem egyezik a tároló CA kulcsával: %s"

#: ../yum/__init__.py:5768
msgid "GPG key signature verified against CA Key(s)"
msgstr "A GPG kulcs sikeresen ellenőrizve lett a CA kulcsokkal"

#: ../yum/__init__.py:5776
#, python-format
msgid "Invalid GPG Key from %s: %s"
msgstr "Hibás GPG kulcs a következőből: %s: %s"

#: ../yum/__init__.py:5785
#, python-format
msgid "GPG key parsing failed: key does not have value %s"
msgstr "GPG kulcs elemzés meghiúsult: a kulcs nem tartalmaz értéket: %s"

#: ../yum/__init__.py:5801
#, python-format
msgid ""
"Importing %s key 0x%s:\n"
" Userid     : \"%s\"\n"
" Fingerprint: %s\n"
" Package    : %s (%s)\n"
" From       : %s"
msgstr "A(z) %s kulcs importálása 0x%s:\n Felhasználó: \"%s\"\n Ujjlenyomat: %s\n Csomag       : %s (%s)\n Származás  : %s"

#: ../yum/__init__.py:5811
#, python-format
msgid ""
"Importing %s key 0x%s:\n"
" Userid     : \"%s\"\n"
" Fingerprint: %s\n"
" From       : %s"
msgstr "A(z) %s kulcs importálása 0x%s:\n Felhasználó: \"%s\"\n Ujjlenyomat: %s\n Származás  : %s"

#: ../yum/__init__.py:5839
#, python-format
msgid ""
"\n"
"\n"
"\n"
" Failing package is: %s\n"
" GPG Keys are configured as: %s\n"
msgstr "\n\n\nSikertelen csomag: %s\nGPG kulcsok beállításai: %s\n"

#: ../yum/__init__.py:5853
#, python-format
msgid "GPG key at %s (0x%s) is already installed"
msgstr "A következő GPG kulcs már telepítve van: %s (0x%s)"

#: ../yum/__init__.py:5891
#, python-format
msgid "Key import failed (code %d)"
msgstr "A kulcs importálása meghiúsult (hibakód %d)"

#: ../yum/__init__.py:5893 ../yum/__init__.py:5994
msgid "Key imported successfully"
msgstr "A kulcs importálása sikeres"

#: ../yum/__init__.py:5897
msgid "Didn't install any keys"
msgstr "Nem lett telepítve egyetlen kulcs sem"

#: ../yum/__init__.py:5900
#, python-format
msgid ""
"The GPG keys listed for the \"%s\" repository are already installed but they are not correct for this package.\n"
"Check that the correct key URLs are configured for this repository."
msgstr "A GPG kulcsok a(z) \"%s\" nevű tárolóhoz már telepítve vannak, de nem jók ehhez a csomaghoz.\nKérjük, ellenőrizze, hogy az URL címek helyesen vannak-e megadva ezen tárolóhoz."

#: ../yum/__init__.py:5910
msgid "Import of key(s) didn't help, wrong key(s)?"
msgstr "A kulcsok importálása nem segített, rossz kulcsok?"

#: ../yum/__init__.py:5932
msgid "No"
msgstr "Nem"

#: ../yum/__init__.py:5934
msgid "Yes"
msgstr "Igen"

#: ../yum/__init__.py:5935
#, python-format
msgid ""
"\n"
"\n"
"\n"
" CA Key: %s\n"
" Failing repo is: %s\n"
" GPG Keys are configured as: %s\n"
msgstr "\n\n\nCA Kulcs: %s\nSikertelen tároló: %s\nGPG kulcsok beállításai: %s\n"

#: ../yum/__init__.py:5948
#, python-format
msgid "GPG key at %s (0x%s) is already imported"
msgstr "A következő GPG kulcs már telepítve van: %s (0x%s)"

#: ../yum/__init__.py:5992
#, python-format
msgid "Key %s import failed"
msgstr "%s kulcs importálása sikertelen"

#: ../yum/__init__.py:6009
#, python-format
msgid "Didn't install any keys for repo %s"
msgstr "Nem lett telepítve egyetlen kulcs sem a következő tárolóhoz: %s"

#: ../yum/__init__.py:6014
#, python-format
msgid ""
"The GPG keys listed for the \"%s\" repository are already installed but they are not correct.\n"
"Check that the correct key URLs are configured for this repository."
msgstr "A GPG kulcsok a(z) \"%s\" nevű tárolóhoz már telepítve vannak, de nem jók.\nKérjük, ellenőrizze, hogy az URL címek helyesen vannak-e megadva ezen tárolóhoz."

#: ../yum/__init__.py:6172
msgid "Unable to find a suitable mirror."
msgstr "Nem található megfelelő tükörszerver."

#: ../yum/__init__.py:6174
msgid "Errors were encountered while downloading packages."
msgstr "Hiba történt a csomagok letöltése közben."

#: ../yum/__init__.py:6229
#, python-format
msgid "Please report this error at %s"
msgstr "Kérjük, jelentse ezt a hibát a következő címen: %s"

#: ../yum/__init__.py:6246
msgid "Test Transaction Errors: "
msgstr "Tranzakció teszt hibák: "

#: ../yum/__init__.py:6358
#, python-format
msgid "Could not set cachedir: %s"
msgstr "Gyorsítótár-mappa beállítása meghiúsult: %s"

#: ../yum/__init__.py:6420 ../yum/__init__.py:6422
msgid "Dependencies not solved. Will not save unresolved transaction."
msgstr "A függőségek feloldása még nincs kész. Nem lehet a függőben lévő tranzakciót menteni."

#: ../yum/__init__.py:6455 ../yum/__init__.py:6457
#, python-format
msgid "Could not save transaction file %s: %s"
msgstr "A következő tranzakciófájl mentése meghiúsult: %s: %s"

#: ../yum/__init__.py:6483
#, python-format
msgid "Could not access/read saved transaction %s : %s"
msgstr "Az elmentett tranzakció beolvasása meghiúsult: %s: %s"

#: ../yum/__init__.py:6521
msgid "rpmdb ver mismatched saved transaction version,"
msgstr "rpmdb ver eltér a mentett tranzakció verziójától,"

#: ../yum/__init__.py:6535
msgid "cannot find tsflags or tsflags not integer."
msgstr "Nem található a tsflags, vagy a tsflags nem egész szám."

#: ../yum/__init__.py:6584
#, python-format
msgid "Found txmbr in unknown current state: %s"
msgstr "txmbr található a jelenleg ismeretlen állapotban: %s"

#: ../yum/__init__.py:6588
#, python-format
msgid "Could not find txmbr: %s in state %s"
msgstr "Nem található txmbr: %s állapotban %s"

#: ../yum/__init__.py:6625 ../yum/__init__.py:6642
#, python-format
msgid "Could not find txmbr: %s from origin: %s"
msgstr "Nem található txmbr: %s, innen: %s"

#: ../yum/__init__.py:6667
msgid "Transaction members, relations are missing or ts has been modified,"
msgstr "A tranzakció elemek, tranzakció-viszonyok hiányoznak, vagy a ts feltehetően megváltozott,"

#: ../yum/__init__.py:6670
msgid " ignoring, as requested. You must redepsolve!"
msgstr " átugorva, ahogy kérte. Szükséges a redepsolve!"

#. Debugging output
#: ../yum/__init__.py:6738 ../yum/__init__.py:6757
#, python-format
msgid "%s has been visited already and cannot be removed."
msgstr "%s már el van végezve, nem eltávolítható."

#. Debugging output
#: ../yum/__init__.py:6741
#, python-format
msgid "Examining revdeps of %s"
msgstr "A következő revdeps elemzése: %s"

#. Debugging output
#: ../yum/__init__.py:6762
#, python-format
msgid "%s has revdep %s which was user-installed."
msgstr "%s revdep felhaszáló által lett telepítve: %s"

#: ../yum/__init__.py:6773 ../yum/__init__.py:6779
#, python-format
msgid "%s is needed by a package to be installed."
msgstr "A(z) %s telepítése szükséges egy csomaghoz."

#. Debugging output
#: ../yum/__init__.py:6793
#, python-format
msgid "%s has no user-installed revdeps."
msgstr "Nem található felhasználó által telepített revdep: %s"

#. Mostly copied from YumOutput._outKeyValFill()
#: ../yum/plugins.py:212
msgid "Loaded plugins: "
msgstr "Betöltött bővítmények: "

#: ../yum/plugins.py:226 ../yum/plugins.py:232
#, python-format
msgid "No plugin match for: %s"
msgstr "Nem találhatóak bővítmények a következőre: %s"

#: ../yum/plugins.py:262
#, python-format
msgid "Not loading \"%s\" plugin, as it is disabled"
msgstr "A(z) \"%s\" nevű bővítmény betöltésének kihagyása, mert le van tiltva"

#. Give full backtrace:
#: ../yum/plugins.py:274
#, python-format
msgid "Plugin \"%s\" can't be imported"
msgstr "A(z) \"%s\" nevű bővítmény nem importálható"

#: ../yum/plugins.py:281
#, python-format
msgid "Plugin \"%s\" doesn't specify required API version"
msgstr "A(z) \"%s\" nevű bővítmény nem felel meg a megkövetelt API verziónak"

#: ../yum/plugins.py:286
#, python-format
msgid "Plugin \"%s\" requires API %s. Supported API is %s."
msgstr "A(z) \"%s\" nevű bővítmény megköveteli a %s APIt. Támogatott API: %s."

#: ../yum/plugins.py:319
#, python-format
msgid "Loading \"%s\" plugin"
msgstr "A következő bővítmény betöltése: %s"

#: ../yum/plugins.py:326
#, python-format
msgid "Two or more plugins with the name \"%s\" exist in the plugin search path"
msgstr "Kettő vagy több plugin már léteik a keresési útvonalon a következő néven: %s"

#: ../yum/plugins.py:346
#, python-format
msgid "Configuration file %s not found"
msgstr "A következő konfigurációs fájl nem található: %s"

#. for
#. Configuration files for the plugin not found
#: ../yum/plugins.py:349
#, python-format
msgid "Unable to find configuration file for plugin %s"
msgstr "Nem található konfigurációs fájl a következő bővítményhez: %s"

#: ../yum/plugins.py:553
msgid "registration of commands not supported"
msgstr "parancsok regisztrációja nem támogatott"

#: ../yum/rpmsack.py:159
msgid "has missing requires of"
msgstr "hiányoznak követelmények"

#: ../yum/rpmsack.py:162
msgid "has installed conflicts"
msgstr "konfliktusokat teremtett"

#: ../yum/rpmsack.py:171
#, python-format
msgid "%s is a duplicate with %s"
msgstr "%s egy másolat a következőhöz: %s"

#: ../yum/rpmsack.py:179
#, python-format
msgid "%s is obsoleted by %s"
msgstr "%s már elavult a következő miatt: %s"

#: ../yum/rpmsack.py:187
#, python-format
msgid "%s provides %s but it cannot be found"
msgstr "%s ellátja a következőt: %s, de nem található"

#: ../yum/rpmtrans.py:80
msgid "Repackaging"
msgstr "Újracsomagolás"

#: ../yum/rpmtrans.py:149
#, python-format
msgid "Verify: %u/%u: %s"
msgstr "Ellenőrzés: %u/%u: %s"

#: ../yum/yumRepo.py:919
#, python-format
msgid ""
"Insufficient space in download directory %s\n"
"    * free   %s\n"
"    * needed %s"
msgstr "Nincs elég hely a letöltési könyvtárban: %s\n    * szabad    %s\n    * szükséges %s"

#: ../yum/yumRepo.py:986
msgid "Package does not match intended download."
msgstr "Ez nem az általunk letölteni kívánt csomag."

#: ../rpmUtils/oldUtils.py:33
#, python-format
msgid "Header cannot be opened or does not match %s, %s."
msgstr "A fejléc nem nyitható meg, vagy nem található a következő: %s, %s"

#: ../rpmUtils/oldUtils.py:53
#, python-format
msgid "RPM %s fails md5 check"
msgstr "RPM %s md5 ellenőrzése hibát mutat"

#: ../rpmUtils/oldUtils.py:151
msgid "Could not open RPM database for reading. Perhaps it is already in use?"
msgstr "Nem nyitható meg olvasásra az RPM adatbázis. Elképzelhető, hogy már használatban van."

#: ../rpmUtils/oldUtils.py:183
msgid "Got an empty Header, something has gone wrong"
msgstr "Egy üres fejléc található, valami rossz történhett"

#: ../rpmUtils/oldUtils.py:253 ../rpmUtils/oldUtils.py:260
#: ../rpmUtils/oldUtils.py:263 ../rpmUtils/oldUtils.py:266
#, python-format
msgid "Damaged Header %s"
msgstr "Sérült Fejléc %s"

#: ../rpmUtils/oldUtils.py:281
#, python-format
msgid "Error opening rpm %s - error %s"
msgstr "Hiba a következő rpm megnyitásánál: %s - hiba: %s"
